[2019-01-11 13:45:49 -0500] [10252] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x1046f51b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x1046e7f50>
  pre_fork: <function pre_fork at 0x1046f50c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x1046f5578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x1046f5aa0>
  logconfig: None
  logconfig_dict: {}
  config: gunicorn.conf.py
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x1046e7e60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x1046f5668>
  post_request: <function post_request at 0x1046f56e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x1046f5398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x1046e7d70>
  post_worker_init: <function post_worker_init at 0x1046f52a8>
  worker_abort: <function worker_abort at 0x1046f5488>
  child_exit: <function child_exit at 0x1046f57d0>
  worker_exit: <function worker_exit at 0x1046f58c0>
  default_proc_name: homepage.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['127.0.0.1:9000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x1046f59b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 13:45:49 -0500] [10252] [INFO] Starting gunicorn 19.9.0
[2019-01-11 13:45:49 -0500] [10252] [DEBUG] Arbiter booted
[2019-01-11 13:45:49 -0500] [10252] [INFO] Listening at: http://127.0.0.1:9000 (10252)
[2019-01-11 13:45:49 -0500] [10252] [INFO] Using worker: sync
[2019-01-11 13:45:49 -0500] [10255] [INFO] Booting worker with pid: 10255
[2019-01-11 13:45:49 -0500] [10252] [DEBUG] 1 workers
[2019-01-11 13:45:49 -0500] [10255] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
ImportError: No module named wsgi
[2019-01-11 13:45:49 -0500] [10255] [INFO] Worker exiting (pid: 10255)
[2019-01-11 13:45:49 -0500] [10252] [INFO] Shutting down: Master
[2019-01-11 13:45:49 -0500] [10252] [INFO] Reason: Worker failed to boot.
[2019-01-11 13:49:54 -0500] [10434] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x1070c01b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x1070acf50>
  pre_fork: <function pre_fork at 0x1070c00c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x1070c0578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x1070c0aa0>
  logconfig: None
  logconfig_dict: {}
  config: gunicorn.conf.py
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x1070ace60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x1070c0668>
  post_request: <function post_request at 0x1070c06e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x1070c0398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x1070acd70>
  post_worker_init: <function post_worker_init at 0x1070c02a8>
  worker_abort: <function worker_abort at 0x1070c0488>
  child_exit: <function child_exit at 0x1070c07d0>
  worker_exit: <function worker_exit at 0x1070c08c0>
  default_proc_name: homepage/wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['127.0.0.1:9000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x1070c09b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 13:49:54 -0500] [10434] [INFO] Starting gunicorn 19.9.0
[2019-01-11 13:49:54 -0500] [10434] [DEBUG] Arbiter booted
[2019-01-11 13:49:54 -0500] [10434] [INFO] Listening at: http://127.0.0.1:9000 (10434)
[2019-01-11 13:49:54 -0500] [10434] [INFO] Using worker: sync
[2019-01-11 13:49:54 -0500] [10437] [INFO] Booting worker with pid: 10437
[2019-01-11 13:49:54 -0500] [10437] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
ImportError: Import by filename is not supported.
[2019-01-11 13:49:54 -0500] [10437] [INFO] Worker exiting (pid: 10437)
[2019-01-11 13:49:54 -0500] [10434] [DEBUG] 1 workers
[2019-01-11 13:49:54 -0500] [10434] [INFO] Shutting down: Master
[2019-01-11 13:49:54 -0500] [10434] [INFO] Reason: Worker failed to boot.
[2019-01-11 13:51:33 -0500] [10501] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x107be51b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x107bd1f50>
  pre_fork: <function pre_fork at 0x107be50c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x107be5578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x107be5aa0>
  logconfig: None
  logconfig_dict: {}
  config: gunicorn.conf.py
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x107bd1e60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x107be5668>
  post_request: <function post_request at 0x107be56e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x107be5398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x107bd1d70>
  post_worker_init: <function post_worker_init at 0x107be52a8>
  worker_abort: <function worker_abort at 0x107be5488>
  child_exit: <function child_exit at 0x107be57d0>
  worker_exit: <function worker_exit at 0x107be58c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['127.0.0.1:9000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x107be59b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 13:51:33 -0500] [10501] [INFO] Starting gunicorn 19.9.0
[2019-01-11 13:51:33 -0500] [10501] [DEBUG] Arbiter booted
[2019-01-11 13:51:33 -0500] [10501] [INFO] Listening at: http://127.0.0.1:9000 (10501)
[2019-01-11 13:51:33 -0500] [10501] [INFO] Using worker: sync
[2019-01-11 13:51:33 -0500] [10504] [INFO] Booting worker with pid: 10504
[2019-01-11 13:51:33 -0500] [10501] [DEBUG] 1 workers
[2019-01-11 13:51:35 -0500] [10504] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 13:51:35 -0500] [10504] [INFO] Worker exiting (pid: 10504)
[2019-01-11 13:51:35 -0500] [10501] [INFO] Shutting down: Master
[2019-01-11 13:51:35 -0500] [10501] [INFO] Reason: Worker failed to boot.
[2019-01-11 13:53:49 -0500] [10599] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x109bbf1b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x109babf50>
  pre_fork: <function pre_fork at 0x109bbf0c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x109bbf578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x109bbfaa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x109babe60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x109bbf668>
  post_request: <function post_request at 0x109bbf6e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x109bbf398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x109babd70>
  post_worker_init: <function post_worker_init at 0x109bbf2a8>
  worker_abort: <function worker_abort at 0x109bbf488>
  child_exit: <function child_exit at 0x109bbf7d0>
  worker_exit: <function worker_exit at 0x109bbf8c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['127.0.0.1:9000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x109bbf9b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 13:53:49 -0500] [10599] [INFO] Starting gunicorn 19.9.0
[2019-01-11 13:53:49 -0500] [10599] [DEBUG] Arbiter booted
[2019-01-11 13:53:49 -0500] [10599] [INFO] Listening at: http://127.0.0.1:9000 (10599)
[2019-01-11 13:53:49 -0500] [10599] [INFO] Using worker: sync
[2019-01-11 13:53:49 -0500] [10602] [INFO] Booting worker with pid: 10602
[2019-01-11 13:53:49 -0500] [10599] [DEBUG] 1 workers
[2019-01-11 13:53:49 -0500] [10602] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 13:53:49 -0500] [10602] [INFO] Worker exiting (pid: 10602)
[2019-01-11 13:53:49 -0500] [10599] [INFO] Shutting down: Master
[2019-01-11 13:53:49 -0500] [10599] [INFO] Reason: Worker failed to boot.
[2019-01-11 13:54:55 -0500] [10654] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x1072041b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x1071f0f50>
  pre_fork: <function pre_fork at 0x1072040c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x107204578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x107204aa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x1071f0e60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x107204668>
  post_request: <function post_request at 0x1072046e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x107204398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x1071f0d70>
  post_worker_init: <function post_worker_init at 0x1072042a8>
  worker_abort: <function worker_abort at 0x107204488>
  child_exit: <function child_exit at 0x1072047d0>
  worker_exit: <function worker_exit at 0x1072048c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:9000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x1072049b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 13:54:55 -0500] [10654] [INFO] Starting gunicorn 19.9.0
[2019-01-11 13:54:55 -0500] [10654] [DEBUG] Arbiter booted
[2019-01-11 13:54:55 -0500] [10654] [INFO] Listening at: http://0.0.0.0:9000 (10654)
[2019-01-11 13:54:55 -0500] [10654] [INFO] Using worker: sync
[2019-01-11 13:54:55 -0500] [10657] [INFO] Booting worker with pid: 10657
[2019-01-11 13:54:55 -0500] [10654] [DEBUG] 1 workers
[2019-01-11 13:54:55 -0500] [10657] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 13:54:55 -0500] [10657] [INFO] Worker exiting (pid: 10657)
[2019-01-11 13:54:55 -0500] [10654] [INFO] Shutting down: Master
[2019-01-11 13:54:55 -0500] [10654] [INFO] Reason: Worker failed to boot.
[2019-01-11 13:58:54 -0500] [11755] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x10345d1b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x103449f50>
  pre_fork: <function pre_fork at 0x10345d0c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x10345d578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x10345daa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x103449e60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x10345d668>
  post_request: <function post_request at 0x10345d6e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x10345d398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x103449d70>
  post_worker_init: <function post_worker_init at 0x10345d2a8>
  worker_abort: <function worker_abort at 0x10345d488>
  child_exit: <function child_exit at 0x10345d7d0>
  worker_exit: <function worker_exit at 0x10345d8c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:8000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x10345d9b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 13:58:54 -0500] [11755] [INFO] Starting gunicorn 19.9.0
[2019-01-11 13:58:54 -0500] [11755] [DEBUG] Arbiter booted
[2019-01-11 13:58:54 -0500] [11755] [INFO] Listening at: http://0.0.0.0:8000 (11755)
[2019-01-11 13:58:54 -0500] [11755] [INFO] Using worker: sync
[2019-01-11 13:58:54 -0500] [11758] [INFO] Booting worker with pid: 11758
[2019-01-11 13:58:54 -0500] [11755] [DEBUG] 1 workers
[2019-01-11 13:58:54 -0500] [11758] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 13:58:54 -0500] [11758] [INFO] Worker exiting (pid: 11758)
[2019-01-11 13:58:54 -0500] [11755] [INFO] Shutting down: Master
[2019-01-11 13:58:54 -0500] [11755] [INFO] Reason: Worker failed to boot.
[2019-01-11 13:59:36 -0500] [12037] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x10c7b81b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x10c7a4f50>
  pre_fork: <function pre_fork at 0x10c7b80c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x10c7b8578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x10c7b8aa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x10c7a4e60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x10c7b8668>
  post_request: <function post_request at 0x10c7b86e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x10c7b8398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x10c7a4d70>
  post_worker_init: <function post_worker_init at 0x10c7b82a8>
  worker_abort: <function worker_abort at 0x10c7b8488>
  child_exit: <function child_exit at 0x10c7b87d0>
  worker_exit: <function worker_exit at 0x10c7b88c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:8000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x10c7b89b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 13:59:36 -0500] [12037] [INFO] Starting gunicorn 19.9.0
[2019-01-11 13:59:36 -0500] [12037] [DEBUG] Arbiter booted
[2019-01-11 13:59:36 -0500] [12037] [INFO] Listening at: http://0.0.0.0:8000 (12037)
[2019-01-11 13:59:36 -0500] [12037] [INFO] Using worker: sync
[2019-01-11 13:59:36 -0500] [12042] [INFO] Booting worker with pid: 12042
[2019-01-11 13:59:36 -0500] [12037] [DEBUG] 1 workers
[2019-01-11 13:59:36 -0500] [12042] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 13:59:36 -0500] [12042] [INFO] Worker exiting (pid: 12042)
[2019-01-11 13:59:36 -0500] [12037] [INFO] Shutting down: Master
[2019-01-11 13:59:36 -0500] [12037] [INFO] Reason: Worker failed to boot.
[2019-01-11 13:59:41 -0500] [12073] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x10befd1b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x10bee9f50>
  pre_fork: <function pre_fork at 0x10befd0c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x10befd578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x10befdaa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x10bee9e60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x10befd668>
  post_request: <function post_request at 0x10befd6e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x10befd398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x10bee9d70>
  post_worker_init: <function post_worker_init at 0x10befd2a8>
  worker_abort: <function worker_abort at 0x10befd488>
  child_exit: <function child_exit at 0x10befd7d0>
  worker_exit: <function worker_exit at 0x10befd8c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:8000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x10befd9b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 13:59:41 -0500] [12073] [INFO] Starting gunicorn 19.9.0
[2019-01-11 13:59:41 -0500] [12073] [DEBUG] Arbiter booted
[2019-01-11 13:59:41 -0500] [12073] [INFO] Listening at: http://0.0.0.0:8000 (12073)
[2019-01-11 13:59:41 -0500] [12073] [INFO] Using worker: sync
[2019-01-11 13:59:41 -0500] [12076] [INFO] Booting worker with pid: 12076
[2019-01-11 13:59:41 -0500] [12073] [DEBUG] 1 workers
[2019-01-11 13:59:41 -0500] [12076] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 13:59:41 -0500] [12076] [INFO] Worker exiting (pid: 12076)
[2019-01-11 13:59:41 -0500] [12073] [INFO] Shutting down: Master
[2019-01-11 13:59:41 -0500] [12073] [INFO] Reason: Worker failed to boot.
[2019-01-11 13:59:49 -0500] [12132] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x1068fd1b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x1068e9f50>
  pre_fork: <function pre_fork at 0x1068fd0c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x1068fd578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x1068fdaa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x1068e9e60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x1068fd668>
  post_request: <function post_request at 0x1068fd6e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x1068fd398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x1068e9d70>
  post_worker_init: <function post_worker_init at 0x1068fd2a8>
  worker_abort: <function worker_abort at 0x1068fd488>
  child_exit: <function child_exit at 0x1068fd7d0>
  worker_exit: <function worker_exit at 0x1068fd8c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:8000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x1068fd9b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 13:59:49 -0500] [12132] [INFO] Starting gunicorn 19.9.0
[2019-01-11 13:59:49 -0500] [12132] [DEBUG] Arbiter booted
[2019-01-11 13:59:49 -0500] [12132] [INFO] Listening at: http://0.0.0.0:8000 (12132)
[2019-01-11 13:59:49 -0500] [12132] [INFO] Using worker: sync
[2019-01-11 13:59:49 -0500] [12140] [INFO] Booting worker with pid: 12140
[2019-01-11 13:59:49 -0500] [12132] [DEBUG] 1 workers
[2019-01-11 13:59:49 -0500] [12140] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 13:59:49 -0500] [12140] [INFO] Worker exiting (pid: 12140)
[2019-01-11 13:59:49 -0500] [12132] [INFO] Shutting down: Master
[2019-01-11 13:59:49 -0500] [12132] [INFO] Reason: Worker failed to boot.
[2019-01-11 13:59:53 -0500] [12165] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x1088df1b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x1088cbf50>
  pre_fork: <function pre_fork at 0x1088df0c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x1088df578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x1088dfaa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x1088cbe60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x1088df668>
  post_request: <function post_request at 0x1088df6e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x1088df398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x1088cbd70>
  post_worker_init: <function post_worker_init at 0x1088df2a8>
  worker_abort: <function worker_abort at 0x1088df488>
  child_exit: <function child_exit at 0x1088df7d0>
  worker_exit: <function worker_exit at 0x1088df8c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:8000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x1088df9b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 13:59:53 -0500] [12165] [INFO] Starting gunicorn 19.9.0
[2019-01-11 13:59:53 -0500] [12165] [DEBUG] Arbiter booted
[2019-01-11 13:59:53 -0500] [12165] [INFO] Listening at: http://0.0.0.0:8000 (12165)
[2019-01-11 13:59:53 -0500] [12165] [INFO] Using worker: sync
[2019-01-11 13:59:53 -0500] [12169] [INFO] Booting worker with pid: 12169
[2019-01-11 13:59:53 -0500] [12165] [DEBUG] 1 workers
[2019-01-11 13:59:54 -0500] [12169] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 13:59:54 -0500] [12169] [INFO] Worker exiting (pid: 12169)
[2019-01-11 13:59:54 -0500] [12165] [INFO] Shutting down: Master
[2019-01-11 13:59:54 -0500] [12165] [INFO] Reason: Worker failed to boot.
[2019-01-11 14:00:30 -0500] [12413] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x10c8081b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x10c7f4f50>
  pre_fork: <function pre_fork at 0x10c8080c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x10c808578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x10c808aa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x10c7f4e60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x10c808668>
  post_request: <function post_request at 0x10c8086e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x10c808398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x10c7f4d70>
  post_worker_init: <function post_worker_init at 0x10c8082a8>
  worker_abort: <function worker_abort at 0x10c808488>
  child_exit: <function child_exit at 0x10c8087d0>
  worker_exit: <function worker_exit at 0x10c8088c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:8000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x10c8089b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 14:00:30 -0500] [12413] [INFO] Starting gunicorn 19.9.0
[2019-01-11 14:00:30 -0500] [12413] [DEBUG] Arbiter booted
[2019-01-11 14:00:30 -0500] [12413] [INFO] Listening at: http://0.0.0.0:8000 (12413)
[2019-01-11 14:00:30 -0500] [12413] [INFO] Using worker: sync
[2019-01-11 14:00:30 -0500] [12417] [INFO] Booting worker with pid: 12417
[2019-01-11 14:00:30 -0500] [12413] [DEBUG] 1 workers
[2019-01-11 14:00:30 -0500] [12417] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 14:00:30 -0500] [12417] [INFO] Worker exiting (pid: 12417)
[2019-01-11 14:00:30 -0500] [12413] [INFO] Shutting down: Master
[2019-01-11 14:00:30 -0500] [12413] [INFO] Reason: Worker failed to boot.
[2019-01-11 14:00:33 -0500] [12437] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x10593f1b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x10592bf50>
  pre_fork: <function pre_fork at 0x10593f0c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x10593f578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x10593faa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x10592be60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x10593f668>
  post_request: <function post_request at 0x10593f6e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x10593f398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x10592bd70>
  post_worker_init: <function post_worker_init at 0x10593f2a8>
  worker_abort: <function worker_abort at 0x10593f488>
  child_exit: <function child_exit at 0x10593f7d0>
  worker_exit: <function worker_exit at 0x10593f8c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:8000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x10593f9b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 14:00:33 -0500] [12437] [INFO] Starting gunicorn 19.9.0
[2019-01-11 14:00:33 -0500] [12437] [DEBUG] Arbiter booted
[2019-01-11 14:00:33 -0500] [12437] [INFO] Listening at: http://0.0.0.0:8000 (12437)
[2019-01-11 14:00:33 -0500] [12437] [INFO] Using worker: sync
[2019-01-11 14:00:33 -0500] [12441] [INFO] Booting worker with pid: 12441
[2019-01-11 14:00:33 -0500] [12437] [DEBUG] 1 workers
[2019-01-11 14:00:33 -0500] [12441] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 14:00:33 -0500] [12441] [INFO] Worker exiting (pid: 12441)
[2019-01-11 14:00:33 -0500] [12437] [INFO] Shutting down: Master
[2019-01-11 14:00:33 -0500] [12437] [INFO] Reason: Worker failed to boot.
[2019-01-11 14:00:38 -0500] [12478] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x10ecfe1b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x10eceaf50>
  pre_fork: <function pre_fork at 0x10ecfe0c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x10ecfe578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x10ecfeaa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x10eceae60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x10ecfe668>
  post_request: <function post_request at 0x10ecfe6e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x10ecfe398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x10ecead70>
  post_worker_init: <function post_worker_init at 0x10ecfe2a8>
  worker_abort: <function worker_abort at 0x10ecfe488>
  child_exit: <function child_exit at 0x10ecfe7d0>
  worker_exit: <function worker_exit at 0x10ecfe8c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:8000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x10ecfe9b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 14:00:38 -0500] [12478] [INFO] Starting gunicorn 19.9.0
[2019-01-11 14:00:38 -0500] [12478] [DEBUG] Arbiter booted
[2019-01-11 14:00:38 -0500] [12478] [INFO] Listening at: http://0.0.0.0:8000 (12478)
[2019-01-11 14:00:38 -0500] [12478] [INFO] Using worker: sync
[2019-01-11 14:00:38 -0500] [12486] [INFO] Booting worker with pid: 12486
[2019-01-11 14:00:38 -0500] [12478] [DEBUG] 1 workers
[2019-01-11 14:00:38 -0500] [12486] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 14:00:38 -0500] [12486] [INFO] Worker exiting (pid: 12486)
[2019-01-11 14:00:38 -0500] [12478] [INFO] Shutting down: Master
[2019-01-11 14:00:38 -0500] [12478] [INFO] Reason: Worker failed to boot.
[2019-01-11 14:02:14 -0500] [13114] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x106c421b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x106c2ef50>
  pre_fork: <function pre_fork at 0x106c420c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x106c42578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x106c42aa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x106c2ee60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x106c42668>
  post_request: <function post_request at 0x106c426e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x106c42398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x106c2ed70>
  post_worker_init: <function post_worker_init at 0x106c422a8>
  worker_abort: <function worker_abort at 0x106c42488>
  child_exit: <function child_exit at 0x106c427d0>
  worker_exit: <function worker_exit at 0x106c428c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:8000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x106c429b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 14:02:14 -0500] [13114] [INFO] Starting gunicorn 19.9.0
[2019-01-11 14:02:14 -0500] [13114] [DEBUG] Arbiter booted
[2019-01-11 14:02:14 -0500] [13114] [INFO] Listening at: http://0.0.0.0:8000 (13114)
[2019-01-11 14:02:14 -0500] [13114] [INFO] Using worker: sync
[2019-01-11 14:02:14 -0500] [13124] [INFO] Booting worker with pid: 13124
[2019-01-11 14:02:14 -0500] [13114] [DEBUG] 1 workers
[2019-01-11 14:02:15 -0500] [13124] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 14:02:15 -0500] [13124] [INFO] Worker exiting (pid: 13124)
[2019-01-11 14:02:15 -0500] [13114] [INFO] Shutting down: Master
[2019-01-11 14:02:15 -0500] [13114] [INFO] Reason: Worker failed to boot.
[2019-01-11 14:03:23 -0500] [13575] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x1085e41b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x1085d0f50>
  pre_fork: <function pre_fork at 0x1085e40c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x1085e4578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x1085e4aa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x1085d0e60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x1085e4668>
  post_request: <function post_request at 0x1085e46e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x1085e4398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x1085d0d70>
  post_worker_init: <function post_worker_init at 0x1085e42a8>
  worker_abort: <function worker_abort at 0x1085e4488>
  child_exit: <function child_exit at 0x1085e47d0>
  worker_exit: <function worker_exit at 0x1085e48c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:8000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x1085e49b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 14:03:23 -0500] [13575] [INFO] Starting gunicorn 19.9.0
[2019-01-11 14:03:23 -0500] [13575] [DEBUG] Arbiter booted
[2019-01-11 14:03:23 -0500] [13575] [INFO] Listening at: http://0.0.0.0:8000 (13575)
[2019-01-11 14:03:23 -0500] [13575] [INFO] Using worker: sync
[2019-01-11 14:03:23 -0500] [13578] [INFO] Booting worker with pid: 13578
[2019-01-11 14:03:23 -0500] [13575] [DEBUG] 1 workers
[2019-01-11 14:03:23 -0500] [13578] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 14:03:23 -0500] [13578] [INFO] Worker exiting (pid: 13578)
[2019-01-11 14:03:24 -0500] [13575] [INFO] Shutting down: Master
[2019-01-11 14:03:24 -0500] [13575] [INFO] Reason: Worker failed to boot.
[2019-01-11 14:03:27 -0500] [13600] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x10e5121b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x10e4fef50>
  pre_fork: <function pre_fork at 0x10e5120c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x10e512578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x10e512aa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x10e4fee60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x10e512668>
  post_request: <function post_request at 0x10e5126e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x10e512398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x10e4fed70>
  post_worker_init: <function post_worker_init at 0x10e5122a8>
  worker_abort: <function worker_abort at 0x10e512488>
  child_exit: <function child_exit at 0x10e5127d0>
  worker_exit: <function worker_exit at 0x10e5128c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:8000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x10e5129b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 14:03:27 -0500] [13600] [INFO] Starting gunicorn 19.9.0
[2019-01-11 14:03:27 -0500] [13600] [DEBUG] Arbiter booted
[2019-01-11 14:03:27 -0500] [13600] [INFO] Listening at: http://0.0.0.0:8000 (13600)
[2019-01-11 14:03:27 -0500] [13600] [INFO] Using worker: sync
[2019-01-11 14:03:27 -0500] [13604] [INFO] Booting worker with pid: 13604
[2019-01-11 14:03:27 -0500] [13600] [DEBUG] 1 workers
[2019-01-11 14:03:27 -0500] [13604] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 14:03:27 -0500] [13604] [INFO] Worker exiting (pid: 13604)
[2019-01-11 14:03:27 -0500] [13600] [INFO] Shutting down: Master
[2019-01-11 14:03:27 -0500] [13600] [INFO] Reason: Worker failed to boot.
[2019-01-11 14:03:31 -0500] [13630] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x102bd31b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x102bbff50>
  pre_fork: <function pre_fork at 0x102bd30c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x102bd3578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x102bd3aa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x102bbfe60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x102bd3668>
  post_request: <function post_request at 0x102bd36e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x102bd3398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x102bbfd70>
  post_worker_init: <function post_worker_init at 0x102bd32a8>
  worker_abort: <function worker_abort at 0x102bd3488>
  child_exit: <function child_exit at 0x102bd37d0>
  worker_exit: <function worker_exit at 0x102bd38c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:8000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x102bd39b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 14:03:31 -0500] [13630] [INFO] Starting gunicorn 19.9.0
[2019-01-11 14:03:31 -0500] [13630] [DEBUG] Arbiter booted
[2019-01-11 14:03:31 -0500] [13630] [INFO] Listening at: http://0.0.0.0:8000 (13630)
[2019-01-11 14:03:31 -0500] [13630] [INFO] Using worker: sync
[2019-01-11 14:03:31 -0500] [13639] [INFO] Booting worker with pid: 13639
[2019-01-11 14:03:31 -0500] [13630] [DEBUG] 1 workers
[2019-01-11 14:03:31 -0500] [13639] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 14:03:31 -0500] [13639] [INFO] Worker exiting (pid: 13639)
[2019-01-11 14:03:31 -0500] [13630] [INFO] Shutting down: Master
[2019-01-11 14:03:31 -0500] [13630] [INFO] Reason: Worker failed to boot.
[2019-01-11 14:09:58 -0500] [16164] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x109f091b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x109ef5f50>
  pre_fork: <function pre_fork at 0x109f090c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x109f09578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x109f09aa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x109ef5e60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x109f09668>
  post_request: <function post_request at 0x109f096e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x109f09398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x109ef5d70>
  post_worker_init: <function post_worker_init at 0x109f092a8>
  worker_abort: <function worker_abort at 0x109f09488>
  child_exit: <function child_exit at 0x109f097d0>
  worker_exit: <function worker_exit at 0x109f098c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:8000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x109f099b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 14:09:58 -0500] [16164] [INFO] Starting gunicorn 19.9.0
[2019-01-11 14:09:58 -0500] [16164] [DEBUG] Arbiter booted
[2019-01-11 14:09:58 -0500] [16164] [INFO] Listening at: http://0.0.0.0:8000 (16164)
[2019-01-11 14:09:58 -0500] [16164] [INFO] Using worker: sync
[2019-01-11 14:09:58 -0500] [16174] [INFO] Booting worker with pid: 16174
[2019-01-11 14:09:58 -0500] [16164] [DEBUG] 1 workers
[2019-01-11 14:09:59 -0500] [16174] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named corsheaders
[2019-01-11 14:09:59 -0500] [16174] [INFO] Worker exiting (pid: 16174)
[2019-01-11 14:09:59 -0500] [16164] [INFO] Shutting down: Master
[2019-01-11 14:09:59 -0500] [16164] [INFO] Reason: Worker failed to boot.
[2019-01-11 14:10:03 -0500] [16199] [DEBUG] Current configuration:
  proxy_protocol: False
  worker_connections: 1000
  disable_redirect_access_to_syslog: False
  statsd_host: None
  max_requests_jitter: 0
  post_fork: <function post_fork at 0x10de0c1b8>
  errorlog: /Users/connor/code/website_py2/logs/gunicorn-error.log
  enable_stdio_inheritance: False
  worker_class: sync
  ssl_version: 2
  suppress_ragged_eofs: True
  syslog_facility: user
  when_ready: <function when_ready at 0x10ddf8f50>
  pre_fork: <function pre_fork at 0x10de0c0c8>
  cert_reqs: 0
  preload_app: False
  keepalive: 2
  sendfile: None
  reuse_port: False
  worker_tmp_dir: None
  group: 20
  umask: 0
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  workers: 1
  graceful_timeout: 30
  do_handshake_on_connect: False
  spew: False
  proc_name: None
  pre_exec: <function pre_exec at 0x10de0c578>
  limit_request_fields: 100
  pythonpath: None
  on_exit: <function on_exit at 0x10de0caa0>
  logconfig: None
  logconfig_dict: {}
  config: None
  pidfile: None
  check_config: False
  statsd_prefix:
  on_reload: <function on_reload at 0x10ddf8e60>
  reload_extra_files: []
  accesslog: /Users/connor/code/website_py2/logs/gunicorn-access.log
  reload_engine: auto
  proxy_allow_ips: ['127.0.0.1']
  pre_request: <function pre_request at 0x10de0c668>
  post_request: <function post_request at 0x10de0c6e0>
  user: 502
  forwarded_allow_ips: ['127.0.0.1']
  worker_int: <function worker_int at 0x10de0c398>
  paste: None
  raw_paste_global_conf: []
  threads: 1
  max_requests: 0
  limit_request_line: 4094
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  certfile: None
  on_starting: <function on_starting at 0x10ddf8d70>
  post_worker_init: <function post_worker_init at 0x10de0c2a8>
  worker_abort: <function worker_abort at 0x10de0c488>
  child_exit: <function child_exit at 0x10de0c7d0>
  worker_exit: <function worker_exit at 0x10de0c8c0>
  default_proc_name: website_py2.wsgi
  loglevel: debug
  syslog: False
  syslog_prefix: None
  chdir: /Users/Connor/code/website_py2
  daemon: False
  syslog_addr: unix:///var/run/syslog
  ciphers: TLSv1
  bind: ['0.0.0.0:8000']
  raw_env: []
  initgroups: False
  capture_output: False
  reload: False
  limit_request_field_size: 8190
  nworkers_changed: <function nworkers_changed at 0x10de0c9b0>
  timeout: 30
  keyfile: None
  ca_certs: None
  tmp_upload_dir: None
  backlog: 2048
  logger_class: gunicorn.glogging.Logger
[2019-01-11 14:10:03 -0500] [16199] [INFO] Starting gunicorn 19.9.0
[2019-01-11 14:10:03 -0500] [16199] [DEBUG] Arbiter booted
[2019-01-11 14:10:03 -0500] [16199] [INFO] Listening at: http://0.0.0.0:8000 (16199)
[2019-01-11 14:10:03 -0500] [16199] [INFO] Using worker: sync
[2019-01-11 14:10:03 -0500] [16202] [INFO] Booting worker with pid: 16202
[2019-01-11 14:10:03 -0500] [16199] [DEBUG] 1 workers
[2019-01-11 14:10:03 -0500] [16202] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/Users/Connor/code/website_py2/website_py2/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python2.7/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
[2019-01-12 04:37:05 -0500] [5277] [DEBUG] POST /nuoxi.php
[2019-01-12 04:37:06 -0500] [9117] [DEBUG] POST /godkey.php
[2019-01-12 04:37:07 -0500] [5277] [DEBUG] POST /okokok.php
[2019-01-12 04:37:07 -0500] [5278] [DEBUG] POST /erwa.php
[2019-01-12 04:37:09 -0500] [7554] [DEBUG] POST /pma.php
[2019-01-12 04:37:13 -0500] [7554] [DEBUG] POST /ruyi.php
[2019-01-12 04:37:15 -0500] [9117] [DEBUG] POST /51314.php
[2019-01-12 04:37:16 -0500] [5277] [DEBUG] POST /5201314.php
[2019-01-12 04:37:17 -0500] [5277] [DEBUG] POST /fusheng.php
[2019-01-12 04:37:18 -0500] [9117] [DEBUG] POST /general.php
[2019-01-12 04:37:19 -0500] [9117] [DEBUG] POST /repeat.php
[2019-01-12 04:37:20 -0500] [9117] [DEBUG] POST /ldw.php
[2019-01-12 04:37:21 -0500] [9117] [DEBUG] POST /api.php
[2019-01-12 04:37:21 -0500] [5277] [DEBUG] POST /s1.php
[2019-01-12 04:37:22 -0500] [5278] [DEBUG] POST /xiaodai.php
[2019-01-12 04:37:23 -0500] [7554] [DEBUG] POST /hello.php
[2019-01-12 04:37:24 -0500] [5277] [DEBUG] POST /hello.php
[2019-01-12 04:37:25 -0500] [5278] [DEBUG] POST /admn.php
[2019-01-12 04:37:26 -0500] [5277] [DEBUG] POST /hell.php
[2019-01-12 04:37:26 -0500] [9117] [DEBUG] POST /xp.php
[2019-01-12 04:37:27 -0500] [5277] [DEBUG] POST /1.php
[2019-01-12 04:37:28 -0500] [7554] [DEBUG] POST /2.php
[2019-01-12 04:37:28 -0500] [7554] [DEBUG] POST /p.php
[2019-01-12 04:37:29 -0500] [9117] [DEBUG] POST /1.php
[2019-01-12 04:37:30 -0500] [5278] [DEBUG] POST /a.php
[2019-01-12 04:37:31 -0500] [7554] [DEBUG] POST /m.php
[2019-01-12 04:37:31 -0500] [9117] [DEBUG] POST /conf.php
[2019-01-12 04:37:32 -0500] [5278] [DEBUG] POST /123.php
[2019-01-12 04:37:32 -0500] [5278] [DEBUG] POST /HX.php
[2019-01-12 04:37:33 -0500] [9117] [DEBUG] POST /diy.php
[2019-01-12 04:37:34 -0500] [9117] [DEBUG] POST /666.php
[2019-01-12 04:37:35 -0500] [5277] [DEBUG] POST /777.php
[2019-01-12 04:37:35 -0500] [7554] [DEBUG] POST /qwq.php
[2019-01-12 04:37:36 -0500] [5277] [DEBUG] POST /qwqw.php
[2019-01-12 04:37:37 -0500] [7554] [DEBUG] POST /.php
[2019-01-12 04:37:37 -0500] [9117] [DEBUG] POST /infos.php
[2019-01-12 04:37:39 -0500] [5277] [DEBUG] POST /x.php
[2019-01-12 04:37:40 -0500] [9117] [DEBUG] POST /lucky.php
[2019-01-12 04:37:41 -0500] [9117] [DEBUG] POST /htfr.php
[2019-01-12 04:37:42 -0500] [7554] [DEBUG] POST /zzk.php
[2019-01-12 04:37:44 -0500] [9117] [DEBUG] POST /toor.php
[2019-01-12 04:37:45 -0500] [7554] [DEBUG] POST /uu.php
[2019-01-12 04:37:47 -0500] [5278] [DEBUG] POST /aa.php
[2019-01-12 04:37:48 -0500] [9117] [DEBUG] POST /wb.php
[2019-01-12 04:37:49 -0500] [9117] [DEBUG] POST /yj.php
[2019-01-12 04:37:50 -0500] [5277] [DEBUG] POST /z.php
[2019-01-12 04:37:51 -0500] [9117] [DEBUG] POST /7.php
[2019-01-12 04:37:52 -0500] [9117] [DEBUG] POST /xiaoma.php
[2019-01-12 04:37:53 -0500] [5277] [DEBUG] POST /xiaomae.php
[2019-01-12 04:37:54 -0500] [9117] [DEBUG] POST /xiaomar.php
[2019-01-12 04:37:56 -0500] [7554] [DEBUG] POST /qq.php
[2019-01-12 04:37:57 -0500] [5278] [DEBUG] POST /data.php
[2019-01-12 04:37:58 -0500] [5277] [DEBUG] POST /log.php
[2019-01-12 04:37:58 -0500] [7554] [DEBUG] POST /fack.php
[2019-01-12 04:37:59 -0500] [7554] [DEBUG] POST /angge.php
[2019-01-12 04:38:00 -0500] [7554] [DEBUG] POST /cxfm666.php
[2019-01-12 04:38:01 -0500] [5278] [DEBUG] POST /db.php
[2019-01-12 04:38:01 -0500] [9117] [DEBUG] POST /hacly.php
[2019-01-12 04:38:02 -0500] [5277] [DEBUG] POST /xiaomo.php
[2019-01-12 04:38:03 -0500] [5278] [DEBUG] POST /xiaoyu.php
[2019-01-12 04:38:04 -0500] [5278] [DEBUG] POST /xiaohei.php
[2019-01-12 04:38:05 -0500] [9117] [DEBUG] POST /j.php
[2019-01-12 04:38:06 -0500] [7554] [DEBUG] POST /qq5262.php
[2019-01-12 04:38:06 -0500] [5278] [DEBUG] POST /lost.php
[2019-01-12 04:38:07 -0500] [7554] [DEBUG] POST /php.php
[2019-01-12 04:38:08 -0500] [7554] [DEBUG] POST /win.php
[2019-01-12 04:38:10 -0500] [7554] [DEBUG] POST /win1.php
[2019-01-12 04:38:13 -0500] [9117] [DEBUG] POST /linux.php
[2019-01-12 04:38:14 -0500] [7554] [DEBUG] POST /linux1.php
[2019-01-12 04:38:16 -0500] [5277] [DEBUG] POST /cc.php
[2019-01-12 04:38:18 -0500] [5278] [DEBUG] POST /x.php
[2019-01-12 04:38:19 -0500] [9117] [DEBUG] POST /MCLi.php
[2019-01-12 04:38:20 -0500] [5277] [DEBUG] POST /MCLi.php
[2019-01-12 04:38:21 -0500] [7554] [DEBUG] POST /zxc1.php
[2019-01-12 04:38:21 -0500] [5277] [DEBUG] POST /zxc0.php
[2019-01-12 04:38:22 -0500] [5277] [DEBUG] POST /zxc1.php
[2019-01-12 04:38:23 -0500] [9117] [DEBUG] POST /zxc2.php
[2019-01-12 04:38:24 -0500] [9117] [DEBUG] POST /indexa.php
[2019-01-12 04:38:25 -0500] [5278] [DEBUG] POST /lx.php
[2019-01-12 04:38:26 -0500] [5277] [DEBUG] POST /cn.php
[2019-01-12 04:38:26 -0500] [7554] [DEBUG] POST /api.php
[2019-01-12 04:38:27 -0500] [7554] [DEBUG] POST /index1.php
[2019-01-12 04:38:28 -0500] [9117] [DEBUG] POST /info.php
[2019-01-12 04:38:28 -0500] [5278] [DEBUG] POST /info1.php
[2019-01-12 04:38:29 -0500] [5277] [DEBUG] POST /aaaaaa1.php
[2019-01-12 04:38:30 -0500] [7554] [DEBUG] POST /up.php
[2019-01-12 04:38:30 -0500] [7554] [DEBUG] POST /test123.php
[2019-01-12 04:38:31 -0500] [7554] [DEBUG] POST /fb.php
[2019-01-12 04:38:32 -0500] [9117] [DEBUG] POST /paylog.php
[2019-01-12 04:38:33 -0500] [5277] [DEBUG] POST /paylog.php
[2019-01-12 04:38:33 -0500] [9117] [DEBUG] POST /x.php
[2019-01-12 04:38:34 -0500] [9117] [DEBUG] POST /51.php
[2019-01-12 04:38:35 -0500] [5277] [DEBUG] POST /cadre.php
[2019-01-12 04:38:35 -0500] [5277] [DEBUG] POST /mm.php
[2019-01-12 04:38:36 -0500] [9117] [DEBUG] POST /test.php
[2019-01-12 04:38:37 -0500] [5278] [DEBUG] POST /1q.php
[2019-01-12 04:38:37 -0500] [9117] [DEBUG] POST /1111.php
[2019-01-12 04:38:38 -0500] [5278] [DEBUG] POST /errors.php
[2019-01-12 04:38:40 -0500] [5277] [DEBUG] GET /index.php
[2019-01-12 04:38:41 -0500] [9117] [DEBUG] GET /phpmyadmin/index.php
[2019-01-12 04:38:43 -0500] [5278] [DEBUG] GET /phpMyAdmin/index.php
[2019-01-12 04:38:44 -0500] [7554] [DEBUG] GET /pmd/index.php
[2019-01-12 04:38:45 -0500] [5277] [DEBUG] GET /pma/index.php
[2019-01-12 04:38:46 -0500] [5278] [DEBUG] GET /PMA/index.php
[2019-01-12 04:38:46 -0500] [5277] [DEBUG] GET /PMA2/index.php
[2019-01-12 04:38:47 -0500] [9117] [DEBUG] GET /pmamy/index.php
[2019-01-12 04:38:48 -0500] [5277] [DEBUG] GET /pmamy2/index.php
[2019-01-12 04:38:49 -0500] [5278] [DEBUG] GET /mysql/index.php
[2019-01-12 04:38:49 -0500] [7554] [DEBUG] GET /admin/index.php
[2019-01-12 04:38:50 -0500] [7554] [DEBUG] GET /db/index.php
[2019-01-12 04:38:51 -0500] [7554] [DEBUG] GET /dbadmin/index.php
[2019-01-12 04:38:52 -0500] [5277] [DEBUG] GET /web/phpMyAdmin/index.php
[2019-01-12 04:38:53 -0500] [9117] [DEBUG] GET /admin/pma/index.php
[2019-01-12 04:38:54 -0500] [7554] [DEBUG] GET /admin/PMA/index.php
[2019-01-12 04:38:54 -0500] [5278] [DEBUG] GET /admin/mysql/index.php
[2019-01-12 04:38:55 -0500] [5278] [DEBUG] GET /admin/mysql2/index.php
[2019-01-12 04:38:56 -0500] [9117] [DEBUG] GET /admin/phpmyadmin/index.php
[2019-01-12 04:38:57 -0500] [7554] [DEBUG] GET /admin/phpMyAdmin/index.php
[2019-01-12 04:38:58 -0500] [5277] [DEBUG] GET /admin/phpmyadmin2/index.php
[2019-01-12 04:38:58 -0500] [5277] [DEBUG] GET /mysqladmin/index.php
[2019-01-12 04:38:59 -0500] [5277] [DEBUG] GET /mysql-admin/index.php
[2019-01-12 04:39:00 -0500] [5277] [DEBUG] GET /mysql_admin/index.php
[2019-01-12 04:39:00 -0500] [7554] [DEBUG] GET /phpadmin/index.php
[2019-01-12 04:39:01 -0500] [5278] [DEBUG] GET /phpAdmin/index.php
[2019-01-12 04:39:02 -0500] [9117] [DEBUG] GET /phpmyadmin0/index.php
[2019-01-12 04:39:03 -0500] [5278] [DEBUG] GET /phpmyadmin1/index.php
[2019-01-12 04:39:04 -0500] [5277] [DEBUG] GET /phpmyadmin2/index.php
[2019-01-12 04:39:04 -0500] [7554] [DEBUG] GET /phpMyAdmin-4.4.0/index.php
[2019-01-12 04:39:05 -0500] [9117] [DEBUG] GET /myadmin/index.php
[2019-01-12 04:39:06 -0500] [5277] [DEBUG] GET /myadmin2/index.php
[2019-01-12 04:39:07 -0500] [7554] [DEBUG] GET /xampp/phpmyadmin/index.php
[2019-01-12 04:39:07 -0500] [5278] [DEBUG] GET /phpMyadmin_bak/index.php
[2019-01-12 04:39:08 -0500] [5277] [DEBUG] GET /www/phpMyAdmin/index.php
[2019-01-12 04:39:09 -0500] [7554] [DEBUG] GET /tools/phpMyAdmin/index.php
[2019-01-12 04:39:10 -0500] [5277] [DEBUG] GET /phpmyadmin-old/index.php
[2019-01-12 04:39:10 -0500] [5277] [DEBUG] GET /phpMyAdminold/index.php
[2019-01-12 04:39:11 -0500] [5278] [DEBUG] GET /phpMyAdmin.old/index.php
[2019-01-12 04:39:12 -0500] [7554] [DEBUG] GET /pma-old/index.php
[2019-01-12 04:39:12 -0500] [5278] [DEBUG] GET /claroline/phpMyAdmin/index.php
[2019-01-12 04:39:13 -0500] [7554] [DEBUG] GET /typo3/phpmyadmin/index.php
[2019-01-12 04:39:14 -0500] [5277] [DEBUG] GET /phpma/index.php
[2019-01-12 04:39:14 -0500] [7554] [DEBUG] GET /phpmyadmin/phpmyadmin/index.php
[2019-01-12 04:39:15 -0500] [7554] [DEBUG] GET /phpMyAdmin/phpMyAdmin/index.php
[2019-01-12 04:39:16 -0500] [5278] [DEBUG] GET /phpMyAbmin/index.php
[2019-01-12 04:39:16 -0500] [7554] [DEBUG] GET /phpMyAdmin__/index.php
[2019-01-12 04:39:19 -0500] [7554] [DEBUG] GET /phpMyAdmin+++---/index.php
[2019-01-12 04:39:20 -0500] [5277] [DEBUG] GET /v/index.php
[2019-01-12 04:39:21 -0500] [5278] [DEBUG] GET /phpmyadm1n/index.php
[2019-01-12 04:39:23 -0500] [5278] [DEBUG] GET /phpMyAdm1n/index.php
[2019-01-12 04:39:24 -0500] [9117] [DEBUG] GET /shaAdmin/index.php
[2019-01-12 04:39:25 -0500] [5278] [DEBUG] GET /phpMyadmi/index.php
[2019-01-12 04:39:27 -0500] [7554] [DEBUG] GET /phpMyAdmion/index.php
[2019-01-12 04:39:27 -0500] [7554] [DEBUG] GET /s/index.php
[2019-01-12 04:39:28 -0500] [5277] [DEBUG] GET /MyAdmin/index.php
[2019-01-12 04:39:29 -0500] [5277] [DEBUG] GET /phpMyAdmin1/index.php
[2019-01-12 04:39:30 -0500] [5278] [DEBUG] GET /phpMyAdmin123/index.php
[2019-01-12 04:39:31 -0500] [7554] [DEBUG] GET /pwd/index.php
[2019-01-12 04:39:31 -0500] [5277] [DEBUG] GET /phpMyAdmina/index.php
[2019-01-12 04:39:32 -0500] [9117] [DEBUG] GET /phpMydmin/index.php
[2019-01-12 04:39:33 -0500] [7554] [DEBUG] GET /program/index.php
[2019-01-12 04:39:34 -0500] [9117] [DEBUG] GET /shopdb/index.php
[2019-01-12 04:39:34 -0500] [5278] [DEBUG] GET /phppma/index.php
[2019-01-12 04:39:35 -0500] [7554] [DEBUG] GET /phpmy/index.php
[2019-01-12 04:39:36 -0500] [7554] [DEBUG] GET /mysql/admin/index.php
[2019-01-12 04:39:36 -0500] [5277] [DEBUG] GET /mysql/dbadmin/index.php
[2019-01-12 04:39:37 -0500] [5278] [DEBUG] GET /mysql/sqlmanager/index.php
[2019-01-12 04:39:38 -0500] [5278] [DEBUG] GET /mysql/mysqlmanager/index.php
[2019-01-12 04:39:39 -0500] [9117] [DEBUG] GET /wp-content/plugins/portable-phpmyadmin/wp-pma-mod/index.php
[2019-01-12 04:39:40 -0500] [5277] [DEBUG] GET /manager/html
[2019-01-12 05:11:59 -0500] [7554] [DEBUG] GET /
[2019-01-12 06:16:28 -0500] [9117] [DEBUG] GET /
[2019-01-12 06:16:28 -0500] [5278] [DEBUG] GET /
[2019-01-12 07:31:36 -0500] [9117] [DEBUG] GET /
[2019-01-12 07:31:43 -0500] [9117] [DEBUG] Ignoring EPIPE
[2019-01-12 07:51:40 -0500] [5278] [DEBUG] GET /
[2019-01-12 07:51:44 -0500] [5277] [DEBUG] GET /
[2019-01-12 07:52:03 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:5277)
[2019-01-12 07:52:24 -0500] [16607] [INFO] Booting worker with pid: 16607
[2019-01-12 07:52:49 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16607)
[2019-01-12 07:52:53 -0500] [16620] [INFO] Booting worker with pid: 16620
[2019-01-12 07:53:22 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16620)
[2019-01-12 07:53:24 -0500] [16639] [INFO] Booting worker with pid: 16639
[2019-01-12 07:53:53 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16639)
[2019-01-12 07:53:54 -0500] [16639] [INFO] Worker exiting (pid: 16639)
[2019-01-12 07:53:55 -0500] [16654] [INFO] Booting worker with pid: 16654
[2019-01-12 07:54:25 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16654)
[2019-01-12 07:54:25 -0500] [16654] [INFO] Worker exiting (pid: 16654)
[2019-01-12 07:54:26 -0500] [16673] [INFO] Booting worker with pid: 16673
[2019-01-12 07:54:56 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16673)
[2019-01-12 07:54:56 -0500] [16673] [INFO] Worker exiting (pid: 16673)
[2019-01-12 07:54:57 -0500] [16696] [INFO] Booting worker with pid: 16696
[2019-01-12 07:55:27 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16696)
[2019-01-12 07:55:27 -0500] [16696] [INFO] Worker exiting (pid: 16696)
[2019-01-12 07:55:28 -0500] [16711] [INFO] Booting worker with pid: 16711
[2019-01-12 07:55:58 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16711)
[2019-01-12 07:56:00 -0500] [16727] [INFO] Booting worker with pid: 16727
[2019-01-12 07:56:30 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16727)
[2019-01-12 07:56:31 -0500] [16727] [INFO] Worker exiting (pid: 16727)
[2019-01-12 07:56:32 -0500] [16750] [INFO] Booting worker with pid: 16750
[2019-01-12 07:57:02 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16750)
[2019-01-12 07:57:02 -0500] [16750] [INFO] Worker exiting (pid: 16750)
[2019-01-12 07:57:03 -0500] [16767] [INFO] Booting worker with pid: 16767
[2019-01-12 07:57:33 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16767)
[2019-01-12 07:57:33 -0500] [16767] [INFO] Worker exiting (pid: 16767)
[2019-01-12 07:57:34 -0500] [16785] [INFO] Booting worker with pid: 16785
[2019-01-12 07:58:04 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16785)
[2019-01-12 07:58:05 -0500] [16804] [INFO] Booting worker with pid: 16804
[2019-01-12 07:58:35 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16804)
[2019-01-12 07:58:36 -0500] [16830] [INFO] Booting worker with pid: 16830
[2019-01-12 07:59:06 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16830)
[2019-01-12 07:59:07 -0500] [16830] [INFO] Worker exiting (pid: 16830)
[2019-01-12 07:59:08 -0500] [16849] [INFO] Booting worker with pid: 16849
[2019-01-12 07:59:38 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16849)
[2019-01-12 07:59:39 -0500] [16849] [INFO] Worker exiting (pid: 16849)
[2019-01-12 07:59:40 -0500] [16878] [INFO] Booting worker with pid: 16878
[2019-01-12 08:00:10 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16878)
[2019-01-12 08:00:10 -0500] [16878] [INFO] Worker exiting (pid: 16878)
[2019-01-12 08:00:11 -0500] [16897] [INFO] Booting worker with pid: 16897
[2019-01-12 08:00:41 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16897)
[2019-01-12 08:00:41 -0500] [16897] [INFO] Worker exiting (pid: 16897)
[2019-01-12 08:00:42 -0500] [16913] [INFO] Booting worker with pid: 16913
[2019-01-12 08:01:12 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16913)
[2019-01-12 08:01:12 -0500] [16913] [INFO] Worker exiting (pid: 16913)
[2019-01-12 08:01:13 -0500] [16939] [INFO] Booting worker with pid: 16939
[2019-01-12 08:01:44 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16939)
[2019-01-12 08:01:44 -0500] [16939] [INFO] Worker exiting (pid: 16939)
[2019-01-12 08:01:45 -0500] [16956] [INFO] Booting worker with pid: 16956
[2019-01-12 08:02:15 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16956)
[2019-01-12 08:02:17 -0500] [16974] [INFO] Booting worker with pid: 16974
[2019-01-12 08:02:47 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16974)
[2019-01-12 08:02:47 -0500] [16974] [INFO] Worker exiting (pid: 16974)
[2019-01-12 08:02:48 -0500] [16992] [INFO] Booting worker with pid: 16992
[2019-01-12 08:03:18 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:16992)
[2019-01-12 08:03:19 -0500] [16992] [INFO] Worker exiting (pid: 16992)
[2019-01-12 08:03:20 -0500] [17019] [INFO] Booting worker with pid: 17019
[2019-01-12 08:03:50 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:17019)
[2019-01-12 08:03:50 -0500] [17019] [INFO] Worker exiting (pid: 17019)
[2019-01-12 08:03:51 -0500] [17036] [INFO] Booting worker with pid: 17036
[2019-01-12 08:04:11 -0500] [5278] [DEBUG] GET /
[2019-01-12 08:04:21 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:17036)
[2019-01-12 08:04:22 -0500] [17036] [INFO] Worker exiting (pid: 17036)
[2019-01-12 08:04:23 -0500] [17068] [INFO] Booting worker with pid: 17068
[2019-01-12 08:04:53 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:17068)
[2019-01-12 08:04:53 -0500] [17068] [INFO] Worker exiting (pid: 17068)
[2019-01-12 08:04:54 -0500] [17080] [INFO] Booting worker with pid: 17080
[2019-01-12 08:05:24 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:17080)
[2019-01-12 08:05:24 -0500] [17080] [INFO] Worker exiting (pid: 17080)
[2019-01-12 08:05:25 -0500] [17106] [INFO] Booting worker with pid: 17106
[2019-01-12 08:05:55 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:17106)
[2019-01-12 08:05:56 -0500] [17106] [INFO] Worker exiting (pid: 17106)
[2019-01-12 08:05:57 -0500] [17123] [INFO] Booting worker with pid: 17123
[2019-01-12 08:44:43 -0500] [7554] [DEBUG] GET /
[2019-01-12 08:45:02 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:7554)
[2019-01-12 08:45:05 -0500] [18521] [INFO] Booting worker with pid: 18521
[2019-01-12 08:45:35 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18521)
[2019-01-12 08:45:35 -0500] [18521] [INFO] Worker exiting (pid: 18521)
[2019-01-12 08:45:36 -0500] [18544] [INFO] Booting worker with pid: 18544
[2019-01-12 08:46:06 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18544)
[2019-01-12 08:46:06 -0500] [18544] [INFO] Worker exiting (pid: 18544)
[2019-01-12 08:46:07 -0500] [18564] [INFO] Booting worker with pid: 18564
[2019-01-12 08:46:38 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18564)
[2019-01-12 08:46:38 -0500] [18564] [INFO] Worker exiting (pid: 18564)
[2019-01-12 08:46:39 -0500] [18584] [INFO] Booting worker with pid: 18584
[2019-01-12 08:47:09 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18584)
[2019-01-12 08:47:10 -0500] [18584] [INFO] Worker exiting (pid: 18584)
[2019-01-12 08:47:12 -0500] [18608] [INFO] Booting worker with pid: 18608
[2019-01-12 08:47:42 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18608)
[2019-01-12 08:47:43 -0500] [18608] [INFO] Worker exiting (pid: 18608)
[2019-01-12 08:47:44 -0500] [18628] [INFO] Booting worker with pid: 18628
[2019-01-12 08:48:14 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18628)
[2019-01-12 08:48:15 -0500] [18628] [INFO] Worker exiting (pid: 18628)
[2019-01-12 08:48:16 -0500] [18648] [INFO] Booting worker with pid: 18648
[2019-01-12 08:48:46 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18648)
[2019-01-12 08:48:46 -0500] [18648] [INFO] Worker exiting (pid: 18648)
[2019-01-12 08:48:47 -0500] [18661] [INFO] Booting worker with pid: 18661
[2019-01-12 08:49:17 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18661)
[2019-01-12 08:49:18 -0500] [18661] [INFO] Worker exiting (pid: 18661)
[2019-01-12 08:49:19 -0500] [18677] [INFO] Booting worker with pid: 18677
[2019-01-12 08:49:49 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18677)
[2019-01-12 08:49:50 -0500] [18677] [INFO] Worker exiting (pid: 18677)
[2019-01-12 08:49:51 -0500] [18696] [INFO] Booting worker with pid: 18696
[2019-01-12 08:50:21 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18696)
[2019-01-12 08:50:22 -0500] [18696] [INFO] Worker exiting (pid: 18696)
[2019-01-12 08:50:23 -0500] [18712] [INFO] Booting worker with pid: 18712
[2019-01-12 08:50:53 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18712)
[2019-01-12 08:50:54 -0500] [18712] [INFO] Worker exiting (pid: 18712)
[2019-01-12 08:50:55 -0500] [18733] [INFO] Booting worker with pid: 18733
[2019-01-12 08:51:25 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18733)
[2019-01-12 08:51:25 -0500] [18733] [INFO] Worker exiting (pid: 18733)
[2019-01-12 08:51:26 -0500] [18753] [INFO] Booting worker with pid: 18753
[2019-01-12 08:51:56 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18753)
[2019-01-12 08:51:57 -0500] [18753] [INFO] Worker exiting (pid: 18753)
[2019-01-12 08:51:58 -0500] [18770] [INFO] Booting worker with pid: 18770
[2019-01-12 08:52:28 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18770)
[2019-01-12 08:52:29 -0500] [18770] [INFO] Worker exiting (pid: 18770)
[2019-01-12 08:52:30 -0500] [18788] [INFO] Booting worker with pid: 18788
[2019-01-12 08:53:00 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18788)
[2019-01-12 08:53:01 -0500] [18788] [INFO] Worker exiting (pid: 18788)
[2019-01-12 08:53:02 -0500] [18807] [INFO] Booting worker with pid: 18807
[2019-01-12 08:53:32 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18807)
[2019-01-12 08:53:32 -0500] [18807] [INFO] Worker exiting (pid: 18807)
[2019-01-12 08:53:33 -0500] [18825] [INFO] Booting worker with pid: 18825
[2019-01-12 08:54:04 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18825)
[2019-01-12 08:54:05 -0500] [18825] [INFO] Worker exiting (pid: 18825)
[2019-01-12 08:54:06 -0500] [18841] [INFO] Booting worker with pid: 18841
[2019-01-12 08:54:36 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18841)
[2019-01-12 08:54:36 -0500] [18841] [INFO] Worker exiting (pid: 18841)
[2019-01-12 08:54:38 -0500] [18863] [INFO] Booting worker with pid: 18863
[2019-01-12 08:55:08 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18863)
[2019-01-12 08:55:09 -0500] [18863] [INFO] Worker exiting (pid: 18863)
[2019-01-12 08:55:15 -0500] [18879] [INFO] Booting worker with pid: 18879
[2019-01-12 08:55:43 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18879)
[2019-01-12 08:55:44 -0500] [18879] [INFO] Worker exiting (pid: 18879)
[2019-01-12 08:55:45 -0500] [18893] [INFO] Booting worker with pid: 18893
[2019-01-12 08:55:50 -0500] [17123] [DEBUG] GET /
[2019-01-12 08:56:15 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18893)
[2019-01-12 08:56:16 -0500] [18893] [INFO] Worker exiting (pid: 18893)
[2019-01-12 08:56:17 -0500] [18908] [INFO] Booting worker with pid: 18908
[2019-01-12 08:56:47 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18908)
[2019-01-12 08:56:47 -0500] [18908] [INFO] Worker exiting (pid: 18908)
[2019-01-12 08:56:48 -0500] [18932] [INFO] Booting worker with pid: 18932
[2019-01-12 08:57:19 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18932)
[2019-01-12 08:57:19 -0500] [18932] [INFO] Worker exiting (pid: 18932)
[2019-01-12 08:57:20 -0500] [18949] [INFO] Booting worker with pid: 18949
[2019-01-12 08:57:50 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18949)
[2019-01-12 08:57:51 -0500] [18949] [INFO] Worker exiting (pid: 18949)
[2019-01-12 08:57:52 -0500] [18965] [INFO] Booting worker with pid: 18965
[2019-01-12 08:58:22 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18965)
[2019-01-12 08:58:22 -0500] [18965] [INFO] Worker exiting (pid: 18965)
[2019-01-12 08:58:23 -0500] [18981] [INFO] Booting worker with pid: 18981
[2019-01-12 08:58:53 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:18981)
[2019-01-12 08:58:53 -0500] [18981] [INFO] Worker exiting (pid: 18981)
[2019-01-12 08:58:54 -0500] [19006] [INFO] Booting worker with pid: 19006
[2019-01-12 08:59:24 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19006)
[2019-01-12 08:59:25 -0500] [19006] [INFO] Worker exiting (pid: 19006)
[2019-01-12 08:59:26 -0500] [19025] [INFO] Booting worker with pid: 19025
[2019-01-12 08:59:57 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19025)
[2019-01-12 08:59:57 -0500] [19025] [INFO] Worker exiting (pid: 19025)
[2019-01-12 08:59:58 -0500] [19044] [INFO] Booting worker with pid: 19044
[2019-01-12 09:00:28 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19044)
[2019-01-12 09:00:28 -0500] [19044] [INFO] Worker exiting (pid: 19044)
[2019-01-12 09:00:29 -0500] [19070] [INFO] Booting worker with pid: 19070
[2019-01-12 09:00:59 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19070)
[2019-01-12 09:00:59 -0500] [19070] [INFO] Worker exiting (pid: 19070)
[2019-01-12 09:01:00 -0500] [19087] [INFO] Booting worker with pid: 19087
[2019-01-12 09:01:30 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19087)
[2019-01-12 09:01:30 -0500] [19087] [INFO] Worker exiting (pid: 19087)
[2019-01-12 09:01:31 -0500] [19100] [INFO] Booting worker with pid: 19100
[2019-01-12 09:02:01 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19100)
[2019-01-12 09:02:01 -0500] [19100] [INFO] Worker exiting (pid: 19100)
[2019-01-12 09:02:02 -0500] [19125] [INFO] Booting worker with pid: 19125
[2019-01-12 09:02:33 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19125)
[2019-01-12 09:02:33 -0500] [19125] [INFO] Worker exiting (pid: 19125)
[2019-01-12 09:02:34 -0500] [19143] [INFO] Booting worker with pid: 19143
[2019-01-12 09:03:04 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19143)
[2019-01-12 09:03:04 -0500] [19143] [INFO] Worker exiting (pid: 19143)
[2019-01-12 09:03:05 -0500] [19166] [INFO] Booting worker with pid: 19166
[2019-01-12 09:03:35 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19166)
[2019-01-12 09:03:35 -0500] [19166] [INFO] Worker exiting (pid: 19166)
[2019-01-12 09:03:36 -0500] [19186] [INFO] Booting worker with pid: 19186
[2019-01-12 09:04:07 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19186)
[2019-01-12 09:04:07 -0500] [19186] [INFO] Worker exiting (pid: 19186)
[2019-01-12 09:04:11 -0500] [19211] [INFO] Booting worker with pid: 19211
[2019-01-12 09:04:40 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19211)
[2019-01-12 09:04:41 -0500] [19211] [INFO] Worker exiting (pid: 19211)
[2019-01-12 09:04:42 -0500] [19226] [INFO] Booting worker with pid: 19226
[2019-01-12 09:05:12 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19226)
[2019-01-12 09:05:12 -0500] [19226] [INFO] Worker exiting (pid: 19226)
[2019-01-12 09:05:14 -0500] [19251] [INFO] Booting worker with pid: 19251
[2019-01-12 09:05:45 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19251)
[2019-01-12 09:05:45 -0500] [19251] [INFO] Worker exiting (pid: 19251)
[2019-01-12 09:05:46 -0500] [19274] [INFO] Booting worker with pid: 19274
[2019-01-12 09:06:17 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19274)
[2019-01-12 09:06:17 -0500] [19274] [INFO] Worker exiting (pid: 19274)
[2019-01-12 09:06:18 -0500] [19287] [INFO] Booting worker with pid: 19287
[2019-01-12 09:06:48 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19287)
[2019-01-12 09:06:49 -0500] [19287] [INFO] Worker exiting (pid: 19287)
[2019-01-12 09:06:50 -0500] [19310] [INFO] Booting worker with pid: 19310
[2019-01-12 09:07:20 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19310)
[2019-01-12 09:07:20 -0500] [19310] [INFO] Worker exiting (pid: 19310)
[2019-01-12 09:07:21 -0500] [19327] [INFO] Booting worker with pid: 19327
[2019-01-12 09:07:51 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19327)
[2019-01-12 09:07:51 -0500] [19327] [INFO] Worker exiting (pid: 19327)
[2019-01-12 09:07:52 -0500] [19347] [INFO] Booting worker with pid: 19347
[2019-01-12 09:08:23 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19347)
[2019-01-12 09:08:23 -0500] [19347] [INFO] Worker exiting (pid: 19347)
[2019-01-12 09:08:24 -0500] [19374] [INFO] Booting worker with pid: 19374
[2019-01-12 09:08:54 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19374)
[2019-01-12 09:08:55 -0500] [19374] [INFO] Worker exiting (pid: 19374)
[2019-01-12 09:08:55 -0500] [19399] [INFO] Booting worker with pid: 19399
[2019-01-12 09:09:25 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19399)
[2019-01-12 09:09:25 -0500] [19399] [INFO] Worker exiting (pid: 19399)
[2019-01-12 09:09:26 -0500] [19415] [INFO] Booting worker with pid: 19415
[2019-01-12 09:09:57 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19415)
[2019-01-12 09:09:57 -0500] [19415] [INFO] Worker exiting (pid: 19415)
[2019-01-12 09:09:58 -0500] [19433] [INFO] Booting worker with pid: 19433
[2019-01-12 09:10:28 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19433)
[2019-01-12 09:10:29 -0500] [19433] [INFO] Worker exiting (pid: 19433)
[2019-01-12 09:10:30 -0500] [19452] [INFO] Booting worker with pid: 19452
[2019-01-12 09:11:00 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19452)
[2019-01-12 09:11:01 -0500] [19452] [INFO] Worker exiting (pid: 19452)
[2019-01-12 09:11:02 -0500] [19470] [INFO] Booting worker with pid: 19470
[2019-01-12 09:11:32 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19470)
[2019-01-12 09:11:32 -0500] [19470] [INFO] Worker exiting (pid: 19470)
[2019-01-12 09:11:33 -0500] [19489] [INFO] Booting worker with pid: 19489
[2019-01-12 09:12:03 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19489)
[2019-01-12 09:12:03 -0500] [19489] [INFO] Worker exiting (pid: 19489)
[2019-01-12 09:12:04 -0500] [19506] [INFO] Booting worker with pid: 19506
[2019-01-12 09:12:35 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19506)
[2019-01-12 09:12:35 -0500] [19506] [INFO] Worker exiting (pid: 19506)
[2019-01-12 09:12:36 -0500] [19521] [INFO] Booting worker with pid: 19521
[2019-01-12 09:13:06 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19521)
[2019-01-12 09:13:07 -0500] [19521] [INFO] Worker exiting (pid: 19521)
[2019-01-12 09:13:08 -0500] [19535] [INFO] Booting worker with pid: 19535
[2019-01-12 09:13:38 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19535)
[2019-01-12 09:13:38 -0500] [19535] [INFO] Worker exiting (pid: 19535)
[2019-01-12 09:13:42 -0500] [19551] [INFO] Booting worker with pid: 19551
[2019-01-12 09:14:11 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19551)
[2019-01-12 09:14:12 -0500] [19551] [INFO] Worker exiting (pid: 19551)
[2019-01-12 09:14:14 -0500] [19575] [INFO] Booting worker with pid: 19575
[2019-01-12 09:14:44 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19575)
[2019-01-12 09:14:44 -0500] [19575] [INFO] Worker exiting (pid: 19575)
[2019-01-12 09:14:45 -0500] [19590] [INFO] Booting worker with pid: 19590
[2019-01-12 09:15:15 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19590)
[2019-01-12 09:15:15 -0500] [19590] [INFO] Worker exiting (pid: 19590)
[2019-01-12 09:15:16 -0500] [19610] [INFO] Booting worker with pid: 19610
[2019-01-12 09:15:46 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19610)
[2019-01-12 09:15:47 -0500] [19610] [INFO] Worker exiting (pid: 19610)
[2019-01-12 09:15:48 -0500] [19632] [INFO] Booting worker with pid: 19632
[2019-01-12 09:16:18 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19632)
[2019-01-12 09:16:19 -0500] [19632] [INFO] Worker exiting (pid: 19632)
[2019-01-12 09:16:20 -0500] [19648] [INFO] Booting worker with pid: 19648
[2019-01-12 09:16:50 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19648)
[2019-01-12 09:16:50 -0500] [19648] [INFO] Worker exiting (pid: 19648)
[2019-01-12 09:16:51 -0500] [19674] [INFO] Booting worker with pid: 19674
[2019-01-12 09:17:21 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19674)
[2019-01-12 09:17:21 -0500] [19674] [INFO] Worker exiting (pid: 19674)
[2019-01-12 09:17:22 -0500] [19692] [INFO] Booting worker with pid: 19692
[2019-01-12 09:17:52 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19692)
[2019-01-12 09:17:53 -0500] [19692] [INFO] Worker exiting (pid: 19692)
[2019-01-12 09:17:54 -0500] [19711] [INFO] Booting worker with pid: 19711
[2019-01-12 09:18:24 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19711)
[2019-01-12 09:18:24 -0500] [19711] [INFO] Worker exiting (pid: 19711)
[2019-01-12 09:18:25 -0500] [19730] [INFO] Booting worker with pid: 19730
[2019-01-12 09:18:55 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19730)
[2019-01-12 09:18:55 -0500] [19730] [INFO] Worker exiting (pid: 19730)
[2019-01-12 09:18:56 -0500] [19751] [INFO] Booting worker with pid: 19751
[2019-01-12 09:19:27 -0500] [5201] [CRITICAL] WORKER TIMEOUT (pid:19751)
[2019-01-16 18:23:48 -0500] [22124] [DEBUG] GET /uploader.php
[2019-01-16 18:23:48 -0500] [9117] [DEBUG] GET /cmd.php
[2019-01-16 18:23:49 -0500] [5278] [DEBUG] GET /cmv.php
[2019-01-16 18:24:02 -0500] [22124] [DEBUG] GET /appserv.php
[2019-01-16 18:24:06 -0500] [17123] [DEBUG] GET /phpmyadmin/scripts/setup.php
[2019-01-16 18:24:07 -0500] [9117] [DEBUG] GET /phpMyAdmin/scripts/setup.php
[2019-01-16 18:24:07 -0500] [22124] [DEBUG] GET /phpmyadmin/scripts/db___.init.php
[2019-01-16 18:24:08 -0500] [5278] [DEBUG] GET /phpMyAdmin/scripts/db___.init.php
[2019-01-16 18:24:12 -0500] [22124] [DEBUG] GET /cacti/plugins/weathermap/editor.php
[2019-01-16 18:24:17 -0500] [17123] [DEBUG] POST /xw.php
[2019-01-16 18:24:17 -0500] [22124] [DEBUG] POST /xw1.php
[2019-01-16 18:24:17 -0500] [17123] [DEBUG] POST /9678.php
[2019-01-16 18:24:18 -0500] [5278] [DEBUG] POST /wc.php
[2019-01-16 18:24:24 -0500] [22124] [DEBUG] POST /s.php
[2019-01-16 18:24:24 -0500] [22124] [DEBUG] POST /w.php
[2019-01-16 18:24:25 -0500] [9117] [DEBUG] POST /sheep.php
[2019-01-16 18:24:33 -0500] [17123] [DEBUG] POST /db_session.init.php
[2019-01-16 18:24:34 -0500] [9117] [DEBUG] POST /db__.init.php
[2019-01-16 18:24:34 -0500] [5278] [DEBUG] POST /wp-admins.php
[2019-01-16 18:24:34 -0500] [9117] [DEBUG] POST /m.php
[2019-01-16 18:24:43 -0500] [22124] [DEBUG] POST /mx.php
[2019-01-16 18:24:43 -0500] [5278] [DEBUG] POST /wshell.php
[2019-01-16 18:24:43 -0500] [17123] [DEBUG] POST /xshell.php
[2019-01-16 18:24:44 -0500] [22124] [DEBUG] POST /qq.php
[2019-01-16 18:24:44 -0500] [5278] [DEBUG] POST /conflg.php
[2019-01-16 18:24:44 -0500] [22124] [DEBUG] POST /conflg.php
[2019-01-16 18:24:53 -0500] [9117] [DEBUG] POST /phpStudy.php
[2019-01-16 18:24:53 -0500] [22124] [DEBUG] POST /weixiao.php
[2019-01-16 18:24:54 -0500] [9117] [DEBUG] POST /feixiang.php
[2019-01-16 18:24:54 -0500] [22124] [DEBUG] POST /ak47.php
[2019-01-16 18:24:55 -0500] [9117] [DEBUG] POST /ak48.php
[2019-01-16 18:25:09 -0500] [9117] [DEBUG] GET /phpMyAdmin/index.php
[2019-01-16 18:25:10 -0500] [5278] [DEBUG] GET /pmd/index.php
[2019-01-16 18:25:10 -0500] [22124] [DEBUG] GET /pma/index.php
[2019-01-16 18:25:10 -0500] [9117] [DEBUG] GET /PMA/index.php
[2019-01-16 18:25:11 -0500] [17123] [DEBUG] GET /PMA2/index.php
[2019-01-16 18:25:11 -0500] [5278] [DEBUG] GET /pmamy/index.php
[2019-01-16 18:25:17 -0500] [22124] [DEBUG] GET /mysql/index.php
[2019-01-16 18:25:17 -0500] [17123] [DEBUG] GET /admin/index.php
[2019-01-16 18:25:18 -0500] [5278] [DEBUG] GET /db/index.php
[2019-01-16 18:25:27 -0500] [22124] [DEBUG] GET /admin/pma/index.php
[2019-01-16 18:25:34 -0500] [17123] [DEBUG] GET /admin/mysql2/index.php
[2019-01-16 18:25:43 -0500] [22124] [DEBUG] GET /admin/phpmyadmin2/index.php
[2019-01-16 18:25:44 -0500] [9117] [DEBUG] GET /mysqladmin/index.php
[2019-01-16 18:25:51 -0500] [17123] [DEBUG] GET /phpadmin/index.php
[2019-01-16 18:26:05 -0500] [22124] [DEBUG] GET /phpMyAdmin-4.4.0/index.php
[2019-01-16 18:26:06 -0500] [9117] [DEBUG] GET /myadmin/index.php
[2019-01-16 18:26:06 -0500] [5278] [DEBUG] GET /myadmin2/index.php
[2019-01-16 18:26:07 -0500] [17123] [DEBUG] GET /xampp/phpmyadmin/index.php
[2019-01-16 18:26:10 -0500] [17123] [DEBUG] GET /phpMyadmin_bak/index.php
[2019-01-16 18:26:10 -0500] [5278] [DEBUG] GET /www/phpMyAdmin/index.php
[2019-01-16 18:26:10 -0500] [17123] [DEBUG] GET /tools/phpMyAdmin/index.php
[2019-01-16 18:26:16 -0500] [17123] [DEBUG] GET /phpMyAdminold/index.php
[2019-01-16 18:26:16 -0500] [17123] [DEBUG] GET /phpMyAdmin.old/index.php
[2019-01-16 18:26:17 -0500] [22124] [DEBUG] GET /pma-old/index.php
[2019-01-16 18:26:26 -0500] [22124] [DEBUG] GET /phpma/index.php
[2019-01-16 18:26:27 -0500] [5278] [DEBUG] GET /phpmyadmin/phpmyadmin/index.php
[2019-01-16 18:26:34 -0500] [9117] [DEBUG] GET /phpMyAdmin__/index.php
[2019-01-16 18:26:35 -0500] [22124] [DEBUG] GET /phpMyAdmin+++---/index.php
[2019-01-16 18:26:36 -0500] [17123] [DEBUG] GET /v/index.php
[2019-01-16 18:26:36 -0500] [9117] [DEBUG] GET /phpmyadm1n/index.php
[2019-01-16 18:26:40 -0500] [5278] [DEBUG] GET /shaAdmin/index.php
[2019-01-16 18:26:44 -0500] [22124] [DEBUG] GET /phpMyAdmion/index.php
[2019-01-16 18:26:59 -0500] [22124] [DEBUG] GET /pwd/index.php
[2019-01-16 18:26:59 -0500] [5278] [DEBUG] GET /phpMyAdmina/index.php
[2019-01-16 18:27:00 -0500] [17123] [DEBUG] GET /phpMydmin/index.php
[2019-01-16 18:27:11 -0500] [9117] [DEBUG] GET /phpmy/index.php
[2019-01-16 18:27:12 -0500] [22124] [DEBUG] GET /mysql/admin/index.php
[2019-01-16 18:27:12 -0500] [22124] [DEBUG] GET /mysql/dbadmin/index.php
[2019-01-16 18:27:12 -0500] [17123] [DEBUG] GET /mysql/sqlmanager/index.php
[2019-01-16 18:27:13 -0500] [17123] [DEBUG] GET /mysql/mysqlmanager/index.php
[2019-01-16 18:27:13 -0500] [9117] [DEBUG] GET /wp-content/plugins/portable-phpmyadmin/wp-pma-mod/index.php
[2019-01-16 19:46:21 -0500] [5278] [DEBUG] GET /.git/config
[2019-01-16 20:08:05 -0500] [17123] [DEBUG] GET /
[2019-01-16 20:34:22 -0500] [5278] [DEBUG] GET /
[2019-01-16 21:33:33 -0500] [22124] [DEBUG] GET /
[2019-01-16 21:33:34 -0500] [5278] [DEBUG] GET /favicon.ico
[2019-01-16 21:41:59 -0500] [17123] [DEBUG] GET /
[2019-01-16 22:00:04 -0500] [22124] [DEBUG] GET /.git/config
[2019-01-16 22:29:16 -0500] [5278] [DEBUG] GET /mysql/admin/index.php
[2019-01-16 22:29:19 -0500] [9117] [DEBUG] GET /mysql/dbadmin/index.php
[2019-01-16 22:29:23 -0500] [22124] [DEBUG] GET /mysql/sqlmanager/index.php
[2019-01-16 22:29:25 -0500] [22124] [DEBUG] GET /mysql/mysqlmanager/index.php
[2019-01-16 22:29:26 -0500] [9117] [DEBUG] GET /phpmyadmin/index.php
[2019-01-16 22:29:28 -0500] [5278] [DEBUG] GET /phpMyadmin/index.php
[2019-01-16 22:29:30 -0500] [9117] [DEBUG] GET /phpMyAdmin/index.php
[2019-01-16 22:29:31 -0500] [17123] [DEBUG] GET /phpMyAdmin/index.php
[2019-01-17 00:30:33 -0500] [22124] [DEBUG] GET /.git/config
[2019-01-17 01:15:39 -0500] [9117] [DEBUG] GET /
[2019-01-17 02:18:03 -0500] [5278] [DEBUG] GET /echo.php
[2019-01-17 02:19:40 -0500] [9117] [DEBUG] GET /
[2019-01-17 02:26:43 -0500] [5278] [DEBUG] GET /manager/html
[2019-01-17 02:44:43 -0500] [9117] [DEBUG] HEAD /
[2019-01-17 02:44:43 -0500] [5278] [DEBUG] GET /
[2019-01-17 02:47:01 -0500] [9117] [DEBUG] GET /ccvv
[2019-01-17 03:19:27 -0500] [17123] [DEBUG] HEAD /
[2019-01-17 03:50:51 -0500] [17123] [DEBUG] GET /.git/config
[2019-01-17 04:40:36 -0500] [17123] [DEBUG] GET /
[2019-01-17 05:06:07 -0500] [9117] [DEBUG] GET /
[2019-01-17 05:29:52 -0500] [9117] [DEBUG] GET /mysql/admin/index.php
[2019-01-17 05:29:52 -0500] [9117] [DEBUG] GET /mysql/dbadmin/index.php
[2019-01-17 05:29:53 -0500] [22124] [DEBUG] GET /mysql/sqlmanager/index.php
[2019-01-17 05:29:53 -0500] [17123] [DEBUG] GET /mysql/mysqlmanager/index.php
[2019-01-17 05:29:53 -0500] [22124] [DEBUG] GET /phpmyadmin/index.php
[2019-01-17 05:29:53 -0500] [5278] [DEBUG] GET /phpMyadmin/index.php
[2019-01-17 05:29:54 -0500] [9117] [DEBUG] GET /phpMyAdmin/index.php
[2019-01-17 05:29:54 -0500] [22124] [DEBUG] GET /phpMyAdmin/index.php
[2019-01-17 06:23:58 -0500] [5278] [DEBUG] GET /
[2019-01-17 06:40:59 -0500] [17123] [DEBUG] GET /.git/config
[2019-01-17 06:46:29 -0500] [9117] [DEBUG] GET /phpmyadmin/
[2019-01-17 06:52:58 -0500] [5278] [DEBUG] GET /
[2019-01-17 07:12:02 -0500] [22124] [DEBUG] GET /
[2019-01-17 07:13:12 -0500] [9117] [DEBUG] GET /
[2019-01-17 08:14:57 -0500] [9117] [DEBUG] GET /
[2019-01-17 09:25:40 -0500] [17123] [DEBUG] GET /.git/config
[2019-01-17 10:41:18 -0500] [17123] [DEBUG] GET /.git/config
[2019-01-17 11:43:31 -0500] [17123] [DEBUG] GET /
[2019-01-17 12:29:33 -0500] [9117] [DEBUG] HEAD /
[2019-01-17 12:29:33 -0500] [17123] [DEBUG] GET /clientaccesspolicy.xml
[2019-01-17 12:29:40 -0500] [17123] [DEBUG] GET /
[2019-01-17 13:10:16 -0500] [5278] [DEBUG] GET /
[2019-01-17 14:08:15 -0500] [17123] [DEBUG] GET /
[2019-01-17 14:14:48 -0500] [5278] [DEBUG] GET /.git/config
[2019-01-17 16:01:49 -0500] [5278] [DEBUG] GET /servlet
[2020-03-17 22:26:30 +0900] [65344] [DEBUG] Current configuration:
  config: gunicorn.conf.py
  bind: ['127.0.0.1:8000']
  backlog: 1024
  workers: 4
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  chdir: /Users/connorpro/Code/website_python
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 501
  group: 20
  umask: 0
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /Users/connorpro/Code/website_python/logs/gunicorn-access.log
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /Users/connorpro/Code/website_python/logs/gunicorn-error.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  syslog_addr: unix:///var/run/syslog
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  statsd_prefix:
  proc_name: None
  default_proc_name: website_py2.wsgi
  django_settings: None
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x10ec558c8>
  on_reload: <function OnReload.on_reload at 0x10ec55a60>
  when_ready: <function WhenReady.when_ready at 0x10ec55bf8>
  pre_fork: <function Prefork.pre_fork at 0x10ec55d90>
  post_fork: <function Postfork.post_fork at 0x10ec55f28>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x10ec5e158>
  worker_int: <function WorkerInt.worker_int at 0x10ec5e2f0>
  worker_abort: <function WorkerAbort.worker_abort at 0x10ec5e488>
  pre_exec: <function PreExec.pre_exec at 0x10ec5e620>
  pre_request: <function PreRequest.pre_request at 0x10ec5e7b8>
  post_request: <function PostRequest.post_request at 0x10ec5e8c8>
  worker_exit: <function WorkerExit.worker_exit at 0x10ec5ea60>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x10ec5ebf8>
  on_exit: <function OnExit.on_exit at 0x10ec5ed90>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 3
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
[2020-03-17 22:26:30 +0900] [65344] [INFO] Starting gunicorn 19.6.0
[2020-03-17 22:26:30 +0900] [65344] [DEBUG] Arbiter booted
[2020-03-17 22:26:30 +0900] [65344] [INFO] Listening at: http://127.0.0.1:8000 (65344)
[2020-03-17 22:26:30 +0900] [65344] [INFO] Using worker: sync
[2020-03-17 22:26:30 +0900] [65347] [INFO] Booting worker with pid: 65347
[2020-03-17 22:26:30 +0900] [65349] [INFO] Booting worker with pid: 65349
[2020-03-17 22:26:30 +0900] [65350] [INFO] Booting worker with pid: 65350
[2020-03-17 22:26:30 +0900] [65351] [INFO] Booting worker with pid: 65351
[2020-03-17 22:26:30 +0900] [65344] [DEBUG] 4 workers
[2020-03-17 08:26:47 -0500] [65351] [DEBUG] GET /apidb/ws/chat
[2020-03-17 08:26:47 -0500] [65347] [DEBUG] GET /apidb/comments
[2020-03-18 00:19:01 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:65347)
[2020-03-18 00:19:00 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:65349)
[2020-03-18 00:19:00 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:65350)
[2020-03-17 10:19:00 -0500] [65347] [INFO] Worker exiting (pid: 65347)
[2020-03-18 00:19:00 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:65351)
[2020-03-17 10:19:00 -0500] [65349] [INFO] Worker exiting (pid: 65349)
[2020-03-17 10:19:00 -0500] [65351] [INFO] Worker exiting (pid: 65351)
[2020-03-17 10:19:00 -0500] [65350] [INFO] Worker exiting (pid: 65350)
[2020-03-18 00:19:00 +0900] [21407] [INFO] Booting worker with pid: 21407
[2020-03-18 00:19:00 +0900] [65344] [DEBUG] 1 workers
[2020-03-18 00:19:00 +0900] [21408] [INFO] Booting worker with pid: 21408
[2020-03-18 00:19:00 +0900] [21409] [INFO] Booting worker with pid: 21409
[2020-03-18 00:19:00 +0900] [21410] [INFO] Booting worker with pid: 21410
[2020-03-18 00:19:00 +0900] [65344] [DEBUG] 4 workers
[2020-03-18 00:22:33 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21407)
[2020-03-18 00:22:33 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21408)
[2020-03-18 00:22:33 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21409)
[2020-03-18 00:22:33 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21410)
[2020-03-17 10:22:33 -0500] [21407] [INFO] Worker exiting (pid: 21407)
[2020-03-17 10:22:33 -0500] [21408] [INFO] Worker exiting (pid: 21408)
[2020-03-17 10:22:33 -0500] [21409] [INFO] Worker exiting (pid: 21409)
[2020-03-17 10:22:33 -0500] [21410] [INFO] Worker exiting (pid: 21410)
[2020-03-18 00:22:34 +0900] [21419] [INFO] Booting worker with pid: 21419
[2020-03-18 00:22:34 +0900] [21420] [INFO] Booting worker with pid: 21420
[2020-03-18 00:22:34 +0900] [21422] [INFO] Booting worker with pid: 21422
[2020-03-18 00:22:34 +0900] [21423] [INFO] Booting worker with pid: 21423
[2020-03-18 01:10:49 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21419)
[2020-03-18 01:10:50 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21420)
[2020-03-18 01:10:50 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21422)
[2020-03-18 01:10:50 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21423)
[2020-03-17 11:10:50 -0500] [21419] [INFO] Worker exiting (pid: 21419)
[2020-03-17 11:10:50 -0500] [21420] [INFO] Worker exiting (pid: 21420)
[2020-03-17 11:10:50 -0500] [21422] [INFO] Worker exiting (pid: 21422)
[2020-03-17 11:10:50 -0500] [21423] [INFO] Worker exiting (pid: 21423)
[2020-03-18 01:10:50 +0900] [21447] [INFO] Booting worker with pid: 21447
[2020-03-18 01:10:50 +0900] [65344] [DEBUG] 1 workers
[2020-03-18 01:10:50 +0900] [21449] [INFO] Booting worker with pid: 21449
[2020-03-18 01:10:50 +0900] [21450] [INFO] Booting worker with pid: 21450
[2020-03-18 01:10:50 +0900] [21451] [INFO] Booting worker with pid: 21451
[2020-03-18 01:10:50 +0900] [65344] [DEBUG] 4 workers
[2020-03-18 01:48:40 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21447)
[2020-03-18 01:48:41 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21449)
[2020-03-18 01:48:41 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21450)
[2020-03-18 01:48:41 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21451)
[2020-03-17 11:48:41 -0500] [21449] [INFO] Worker exiting (pid: 21449)
[2020-03-17 11:48:41 -0500] [21450] [INFO] Worker exiting (pid: 21450)
[2020-03-17 11:48:41 -0500] [21447] [INFO] Worker exiting (pid: 21447)
[2020-03-17 11:48:41 -0500] [21451] [INFO] Worker exiting (pid: 21451)
[2020-03-18 01:48:41 +0900] [21470] [INFO] Booting worker with pid: 21470
[2020-03-18 01:48:41 +0900] [65344] [DEBUG] 1 workers
[2020-03-18 01:48:41 +0900] [21472] [INFO] Booting worker with pid: 21472
[2020-03-18 01:48:41 +0900] [21473] [INFO] Booting worker with pid: 21473
[2020-03-18 01:48:41 +0900] [65344] [DEBUG] 4 workers
[2020-03-18 01:48:41 +0900] [21474] [INFO] Booting worker with pid: 21474
[2020-03-18 02:24:47 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21470)
[2020-03-18 02:24:48 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21472)
[2020-03-18 02:24:48 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21473)
[2020-03-18 02:24:48 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21474)
[2020-03-17 12:24:48 -0500] [21470] [INFO] Worker exiting (pid: 21470)
[2020-03-17 12:24:48 -0500] [21473] [INFO] Worker exiting (pid: 21473)
[2020-03-17 12:24:48 -0500] [21472] [INFO] Worker exiting (pid: 21472)
[2020-03-17 12:24:48 -0500] [21474] [INFO] Worker exiting (pid: 21474)
[2020-03-18 02:24:48 +0900] [21481] [INFO] Booting worker with pid: 21481
[2020-03-18 02:24:48 +0900] [21483] [INFO] Booting worker with pid: 21483
[2020-03-18 02:24:48 +0900] [65344] [DEBUG] 2 workers
[2020-03-18 02:24:48 +0900] [21484] [INFO] Booting worker with pid: 21484
[2020-03-18 02:24:48 +0900] [21485] [INFO] Booting worker with pid: 21485
[2020-03-18 02:24:48 +0900] [65344] [DEBUG] 4 workers
[2020-03-18 03:01:44 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21481)
[2020-03-18 03:01:45 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21483)
[2020-03-18 03:01:45 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21484)
[2020-03-18 03:01:45 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21485)
[2020-03-17 13:01:45 -0500] [21481] [INFO] Worker exiting (pid: 21481)
[2020-03-17 13:01:45 -0500] [21483] [INFO] Worker exiting (pid: 21483)
[2020-03-17 13:01:45 -0500] [21484] [INFO] Worker exiting (pid: 21484)
[2020-03-17 13:01:45 -0500] [21485] [INFO] Worker exiting (pid: 21485)
[2020-03-18 03:01:45 +0900] [21490] [INFO] Booting worker with pid: 21490
[2020-03-18 03:01:45 +0900] [65344] [DEBUG] 1 workers
[2020-03-18 03:01:45 +0900] [21492] [INFO] Booting worker with pid: 21492
[2020-03-18 03:01:45 +0900] [21493] [INFO] Booting worker with pid: 21493
[2020-03-18 03:01:45 +0900] [21494] [INFO] Booting worker with pid: 21494
[2020-03-18 03:01:45 +0900] [65344] [DEBUG] 4 workers
[2020-03-18 03:40:15 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21490)
[2020-03-18 03:40:16 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21492)
[2020-03-18 03:40:16 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21493)
[2020-03-18 03:40:16 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21494)
[2020-03-17 13:40:16 -0500] [21490] [INFO] Worker exiting (pid: 21490)
[2020-03-17 13:40:16 -0500] [21493] [INFO] Worker exiting (pid: 21493)
[2020-03-17 13:40:16 -0500] [21492] [INFO] Worker exiting (pid: 21492)
[2020-03-17 13:40:16 -0500] [21494] [INFO] Worker exiting (pid: 21494)
[2020-03-18 03:40:16 +0900] [21515] [INFO] Booting worker with pid: 21515
[2020-03-18 03:40:16 +0900] [65344] [DEBUG] 1 workers
[2020-03-18 03:40:16 +0900] [21516] [INFO] Booting worker with pid: 21516
[2020-03-18 03:40:16 +0900] [21517] [INFO] Booting worker with pid: 21517
[2020-03-18 03:40:16 +0900] [21519] [INFO] Booting worker with pid: 21519
[2020-03-18 03:40:16 +0900] [65344] [DEBUG] 4 workers
[2020-03-18 04:21:52 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21515)
[2020-03-18 04:21:53 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21516)
[2020-03-18 04:21:53 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21517)
[2020-03-18 04:21:53 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21519)
[2020-03-17 14:21:53 -0500] [21516] [INFO] Worker exiting (pid: 21516)
[2020-03-17 14:21:53 -0500] [21517] [INFO] Worker exiting (pid: 21517)
[2020-03-17 14:21:53 -0500] [21515] [INFO] Worker exiting (pid: 21515)
[2020-03-17 14:21:53 -0500] [21519] [INFO] Worker exiting (pid: 21519)
[2020-03-18 04:21:53 +0900] [21557] [INFO] Booting worker with pid: 21557
[2020-03-18 04:21:53 +0900] [65344] [DEBUG] 1 workers
[2020-03-18 04:21:53 +0900] [21558] [INFO] Booting worker with pid: 21558
[2020-03-18 04:21:53 +0900] [21560] [INFO] Booting worker with pid: 21560
[2020-03-18 04:21:53 +0900] [21561] [INFO] Booting worker with pid: 21561
[2020-03-18 04:21:53 +0900] [65344] [DEBUG] 4 workers
[2020-03-18 05:03:31 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21557)
[2020-03-18 05:03:32 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21558)
[2020-03-18 05:03:32 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21560)
[2020-03-18 05:03:32 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21561)
[2020-03-17 15:03:32 -0500] [21557] [INFO] Worker exiting (pid: 21557)
[2020-03-17 15:03:32 -0500] [21558] [INFO] Worker exiting (pid: 21558)
[2020-03-17 15:03:32 -0500] [21560] [INFO] Worker exiting (pid: 21560)
[2020-03-17 15:03:32 -0500] [21561] [INFO] Worker exiting (pid: 21561)
[2020-03-18 05:03:32 +0900] [21584] [INFO] Booting worker with pid: 21584
[2020-03-18 05:03:32 +0900] [65344] [DEBUG] 1 workers
[2020-03-18 05:03:32 +0900] [21586] [INFO] Booting worker with pid: 21586
[2020-03-18 05:03:32 +0900] [21587] [INFO] Booting worker with pid: 21587
[2020-03-18 05:03:32 +0900] [21588] [INFO] Booting worker with pid: 21588
[2020-03-18 05:03:32 +0900] [65344] [DEBUG] 4 workers
[2020-03-18 05:42:22 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21584)
[2020-03-18 05:42:22 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21586)
[2020-03-18 05:42:22 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21587)
[2020-03-18 05:42:22 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21588)
[2020-03-17 15:42:22 -0500] [21586] [INFO] Worker exiting (pid: 21586)
[2020-03-17 15:42:22 -0500] [21584] [INFO] Worker exiting (pid: 21584)
[2020-03-17 15:42:22 -0500] [21587] [INFO] Worker exiting (pid: 21587)
[2020-03-17 15:42:22 -0500] [21588] [INFO] Worker exiting (pid: 21588)
[2020-03-18 05:42:23 +0900] [21592] [INFO] Booting worker with pid: 21592
[2020-03-18 05:42:23 +0900] [21594] [INFO] Booting worker with pid: 21594
[2020-03-18 05:42:23 +0900] [21595] [INFO] Booting worker with pid: 21595
[2020-03-18 05:42:23 +0900] [65344] [DEBUG] 3 workers
[2020-03-18 05:42:23 +0900] [21596] [INFO] Booting worker with pid: 21596
[2020-03-18 05:42:23 +0900] [65344] [DEBUG] 4 workers
[2020-03-18 06:20:53 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21592)
[2020-03-18 06:20:54 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21594)
[2020-03-18 06:20:54 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21595)
[2020-03-18 06:20:54 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21596)
[2020-03-17 16:20:54 -0500] [21592] [INFO] Worker exiting (pid: 21592)
[2020-03-17 16:20:54 -0500] [21594] [INFO] Worker exiting (pid: 21594)
[2020-03-17 16:20:54 -0500] [21595] [INFO] Worker exiting (pid: 21595)
[2020-03-17 16:20:54 -0500] [21596] [INFO] Worker exiting (pid: 21596)
[2020-03-18 06:20:54 +0900] [21606] [INFO] Booting worker with pid: 21606
[2020-03-18 06:20:54 +0900] [65344] [DEBUG] 1 workers
[2020-03-18 06:20:54 +0900] [21608] [INFO] Booting worker with pid: 21608
[2020-03-18 06:20:54 +0900] [21609] [INFO] Booting worker with pid: 21609
[2020-03-18 06:20:54 +0900] [21610] [INFO] Booting worker with pid: 21610
[2020-03-18 06:20:54 +0900] [65344] [DEBUG] 4 workers
[2020-03-18 06:55:46 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21606)
[2020-03-18 06:55:47 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21608)
[2020-03-18 06:55:47 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21609)
[2020-03-18 06:55:47 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21610)
[2020-03-17 16:55:47 -0500] [21606] [INFO] Worker exiting (pid: 21606)
[2020-03-17 16:55:47 -0500] [21608] [INFO] Worker exiting (pid: 21608)
[2020-03-17 16:55:47 -0500] [21609] [INFO] Worker exiting (pid: 21609)
[2020-03-17 16:55:47 -0500] [21610] [INFO] Worker exiting (pid: 21610)
[2020-03-18 06:55:47 +0900] [21631] [INFO] Booting worker with pid: 21631
[2020-03-18 06:55:47 +0900] [21633] [INFO] Booting worker with pid: 21633
[2020-03-18 06:55:47 +0900] [65344] [DEBUG] 2 workers
[2020-03-18 06:55:47 +0900] [21634] [INFO] Booting worker with pid: 21634
[2020-03-18 06:55:47 +0900] [21635] [INFO] Booting worker with pid: 21635
[2020-03-18 06:55:47 +0900] [65344] [DEBUG] 4 workers
[2020-03-18 07:36:18 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21631)
[2020-03-18 07:36:18 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21633)
[2020-03-18 07:36:18 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21634)
[2020-03-18 07:36:18 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21635)
[2020-03-17 17:36:18 -0500] [21631] [INFO] Worker exiting (pid: 21631)
[2020-03-17 17:36:18 -0500] [21634] [INFO] Worker exiting (pid: 21634)
[2020-03-17 17:36:18 -0500] [21633] [INFO] Worker exiting (pid: 21633)
[2020-03-17 17:36:18 -0500] [21635] [INFO] Worker exiting (pid: 21635)
[2020-03-18 07:36:19 +0900] [21654] [INFO] Booting worker with pid: 21654
[2020-03-18 07:36:19 +0900] [21656] [INFO] Booting worker with pid: 21656
[2020-03-18 07:36:19 +0900] [21657] [INFO] Booting worker with pid: 21657
[2020-03-18 07:36:19 +0900] [21658] [INFO] Booting worker with pid: 21658
[2020-03-18 08:20:36 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21654)
[2020-03-18 08:20:36 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21656)
[2020-03-18 08:20:36 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21657)
[2020-03-18 08:20:36 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21658)
[2020-03-17 18:20:36 -0500] [21654] [INFO] Worker exiting (pid: 21654)
[2020-03-17 18:20:36 -0500] [21656] [INFO] Worker exiting (pid: 21656)
[2020-03-17 18:20:36 -0500] [21657] [INFO] Worker exiting (pid: 21657)
[2020-03-17 18:20:36 -0500] [21658] [INFO] Worker exiting (pid: 21658)
[2020-03-18 08:20:37 +0900] [21679] [INFO] Booting worker with pid: 21679
[2020-03-18 08:20:37 +0900] [21681] [INFO] Booting worker with pid: 21681
[2020-03-18 08:20:37 +0900] [21682] [INFO] Booting worker with pid: 21682
[2020-03-18 08:20:37 +0900] [21683] [INFO] Booting worker with pid: 21683
[2020-03-18 08:26:17 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21679)
[2020-03-18 08:26:17 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21681)
[2020-03-18 08:26:17 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21682)
[2020-03-18 08:26:17 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21683)
[2020-03-17 18:26:17 -0500] [21679] [INFO] Worker exiting (pid: 21679)
[2020-03-17 18:26:17 -0500] [21682] [INFO] Worker exiting (pid: 21682)
[2020-03-17 18:26:17 -0500] [21681] [INFO] Worker exiting (pid: 21681)
[2020-03-17 18:26:17 -0500] [21683] [INFO] Worker exiting (pid: 21683)
[2020-03-18 08:26:18 +0900] [21689] [INFO] Booting worker with pid: 21689
[2020-03-18 08:26:18 +0900] [21690] [INFO] Booting worker with pid: 21690
[2020-03-18 08:26:18 +0900] [21692] [INFO] Booting worker with pid: 21692
[2020-03-18 08:26:18 +0900] [21693] [INFO] Booting worker with pid: 21693
[2020-03-18 08:32:40 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21689)
[2020-03-18 08:32:40 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21690)
[2020-03-18 08:32:40 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21692)
[2020-03-18 08:32:40 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21693)
[2020-03-17 18:32:40 -0500] [21690] [INFO] Worker exiting (pid: 21690)
[2020-03-17 18:32:40 -0500] [21692] [INFO] Worker exiting (pid: 21692)
[2020-03-17 18:32:40 -0500] [21689] [INFO] Worker exiting (pid: 21689)
[2020-03-17 18:32:40 -0500] [21693] [INFO] Worker exiting (pid: 21693)
[2020-03-18 08:32:41 +0900] [21706] [INFO] Booting worker with pid: 21706
[2020-03-18 08:32:41 +0900] [21708] [INFO] Booting worker with pid: 21708
[2020-03-18 08:32:41 +0900] [21709] [INFO] Booting worker with pid: 21709
[2020-03-18 08:32:41 +0900] [21710] [INFO] Booting worker with pid: 21710
[2020-03-18 09:06:55 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21706)
[2020-03-18 09:06:55 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21708)
[2020-03-18 09:06:55 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21709)
[2020-03-18 09:06:55 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21710)
[2020-03-17 19:06:55 -0500] [21710] [INFO] Worker exiting (pid: 21710)
[2020-03-17 19:06:55 -0500] [21709] [INFO] Worker exiting (pid: 21709)
[2020-03-17 19:06:55 -0500] [21708] [INFO] Worker exiting (pid: 21708)
[2020-03-17 19:06:55 -0500] [21706] [INFO] Worker exiting (pid: 21706)
[2020-03-18 09:06:56 +0900] [21713] [INFO] Booting worker with pid: 21713
[2020-03-18 09:06:56 +0900] [21715] [INFO] Booting worker with pid: 21715
[2020-03-18 09:06:56 +0900] [21716] [INFO] Booting worker with pid: 21716
[2020-03-18 09:06:56 +0900] [21717] [INFO] Booting worker with pid: 21717
[2020-03-18 10:22:48 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21713)
[2020-03-18 10:22:49 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21715)
[2020-03-18 10:22:49 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21716)
[2020-03-17 20:22:49 -0500] [21713] [INFO] Worker exiting (pid: 21713)
[2020-03-18 10:22:49 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21717)
[2020-03-17 20:22:49 -0500] [21715] [INFO] Worker exiting (pid: 21715)
[2020-03-17 20:22:49 -0500] [21717] [INFO] Worker exiting (pid: 21717)
[2020-03-17 20:22:49 -0500] [21716] [INFO] Worker exiting (pid: 21716)
[2020-03-18 10:22:49 +0900] [21750] [INFO] Booting worker with pid: 21750
[2020-03-18 10:22:49 +0900] [65344] [DEBUG] 1 workers
[2020-03-18 10:22:49 +0900] [21752] [INFO] Booting worker with pid: 21752
[2020-03-18 10:22:49 +0900] [21753] [INFO] Booting worker with pid: 21753
[2020-03-18 10:22:49 +0900] [21754] [INFO] Booting worker with pid: 21754
[2020-03-18 10:22:49 +0900] [65344] [DEBUG] 4 workers
[2020-03-18 10:34:51 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21750)
[2020-03-18 10:34:51 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21752)
[2020-03-18 10:34:51 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21753)
[2020-03-18 10:34:51 +0900] [65344] [CRITICAL] WORKER TIMEOUT (pid:21754)
[2020-03-17 20:34:51 -0500] [21753] [INFO] Worker exiting (pid: 21753)
[2020-03-17 20:34:51 -0500] [21754] [INFO] Worker exiting (pid: 21754)
[2020-03-17 20:34:51 -0500] [21752] [INFO] Worker exiting (pid: 21752)
[2020-03-17 20:34:51 -0500] [21750] [INFO] Worker exiting (pid: 21750)
[2020-03-18 10:34:52 +0900] [21774] [INFO] Booting worker with pid: 21774
[2020-03-18 10:34:52 +0900] [21775] [INFO] Booting worker with pid: 21775
[2020-03-18 10:34:52 +0900] [21777] [INFO] Booting worker with pid: 21777
[2020-03-18 10:34:52 +0900] [21778] [INFO] Booting worker with pid: 21778
[2020-03-17 23:13:09 -0500] [21774] [DEBUG] OPTIONS /apidb/comments
[2020-03-17 23:13:09 -0500] [21775] [DEBUG] GET /apidb/ws/chat
[2020-03-17 23:13:09 -0500] [21774] [DEBUG] GET /apidb/comments
[2020-03-17 23:13:18 -0500] [21778] [DEBUG] OPTIONS /apidb/login
[2020-03-17 23:13:18 -0500] [21775] [DEBUG] POST /apidb/login
[2020-03-17 23:13:23 -0500] [21778] [DEBUG] POST /apidb/login
[2020-03-18 13:13:38 +0900] [65344] [INFO] Handling signal: int
[2020-03-17 23:13:39 -0500] [21774] [INFO] Worker exiting (pid: 21774)
[2020-03-17 23:13:39 -0500] [21775] [INFO] Worker exiting (pid: 21775)
[2020-03-17 23:13:39 -0500] [21778] [INFO] Worker exiting (pid: 21778)
[2020-03-17 23:13:39 -0500] [21777] [INFO] Worker exiting (pid: 21777)
[2020-03-18 13:13:39 +0900] [65344] [INFO] Shutting down: Master
[2020-03-18 23:30:45 +0900] [22890] [DEBUG] Current configuration:
  config: gunicorn.conf.py
  bind: ['127.0.0.1:8000']
  backlog: 1024
  workers: 4
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  chdir: /Users/connorpro/Code/website_python
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 501
  group: 20
  umask: 0
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /Users/connorpro/Code/website_python/logs/gunicorn-access.log
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /Users/connorpro/Code/website_python/logs/gunicorn-error.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  syslog_addr: unix:///var/run/syslog
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  statsd_prefix: 
  proc_name: None
  default_proc_name: website_py2.wsgi
  django_settings: None
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x10506b8c8>
  on_reload: <function OnReload.on_reload at 0x10506ba60>
  when_ready: <function WhenReady.when_ready at 0x10506bbf8>
  pre_fork: <function Prefork.pre_fork at 0x10506bd90>
  post_fork: <function Postfork.post_fork at 0x10506bf28>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x105074158>
  worker_int: <function WorkerInt.worker_int at 0x1050742f0>
  worker_abort: <function WorkerAbort.worker_abort at 0x105074488>
  pre_exec: <function PreExec.pre_exec at 0x105074620>
  pre_request: <function PreRequest.pre_request at 0x1050747b8>
  post_request: <function PostRequest.post_request at 0x1050748c8>
  worker_exit: <function WorkerExit.worker_exit at 0x105074a60>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x105074bf8>
  on_exit: <function OnExit.on_exit at 0x105074d90>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 3
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
[2020-03-18 23:30:45 +0900] [22890] [INFO] Starting gunicorn 19.6.0
[2020-03-18 23:30:45 +0900] [22890] [DEBUG] Arbiter booted
[2020-03-18 23:30:45 +0900] [22890] [INFO] Listening at: http://127.0.0.1:8000 (22890)
[2020-03-18 23:30:45 +0900] [22890] [INFO] Using worker: sync
[2020-03-18 23:30:45 +0900] [22893] [INFO] Booting worker with pid: 22893
[2020-03-18 23:30:46 +0900] [22894] [INFO] Booting worker with pid: 22894
[2020-03-18 23:30:46 +0900] [22895] [INFO] Booting worker with pid: 22895
[2020-03-18 23:30:46 +0900] [22896] [INFO] Booting worker with pid: 22896
[2020-03-18 23:30:46 +0900] [22890] [DEBUG] 4 workers
[2020-03-18 23:31:42 +0900] [22890] [INFO] Handling signal: int
[2020-03-18 09:31:42 -0500] [22894] [INFO] Worker exiting (pid: 22894)
[2020-03-18 09:31:42 -0500] [22895] [INFO] Worker exiting (pid: 22895)
[2020-03-18 09:31:42 -0500] [22896] [INFO] Worker exiting (pid: 22896)
[2020-03-18 09:31:42 -0500] [22893] [INFO] Worker exiting (pid: 22893)
[2020-03-18 23:31:43 +0900] [22890] [INFO] Shutting down: Master
[2020-03-26 12:32:10 -0400] [35690] [DEBUG] Current configuration:
  config: gunicorn.conf.py
  bind: ['127.0.0.1:8000']
  backlog: 1024
  workers: 4
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  chdir: /Users/connorpro/Code/website_python
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 501
  group: 20
  umask: 0
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /Users/connorpro/Code/website_python/logs/gunicorn-access.log
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /Users/connorpro/Code/website_python/logs/gunicorn-error.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  syslog_addr: unix:///var/run/syslog
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  statsd_prefix: 
  proc_name: None
  default_proc_name: website_py2.wsgi
  django_settings: None
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x10e6808c8>
  on_reload: <function OnReload.on_reload at 0x10e680a60>
  when_ready: <function WhenReady.when_ready at 0x10e680bf8>
  pre_fork: <function Prefork.pre_fork at 0x10e680d90>
  post_fork: <function Postfork.post_fork at 0x10e680f28>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x10e689158>
  worker_int: <function WorkerInt.worker_int at 0x10e6892f0>
  worker_abort: <function WorkerAbort.worker_abort at 0x10e689488>
  pre_exec: <function PreExec.pre_exec at 0x10e689620>
  pre_request: <function PreRequest.pre_request at 0x10e6897b8>
  post_request: <function PostRequest.post_request at 0x10e6898c8>
  worker_exit: <function WorkerExit.worker_exit at 0x10e689a60>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x10e689bf8>
  on_exit: <function OnExit.on_exit at 0x10e689d90>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 3
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
[2020-03-26 12:32:10 -0400] [35690] [INFO] Starting gunicorn 19.6.0
[2020-03-26 12:32:10 -0400] [35690] [DEBUG] Arbiter booted
[2020-03-26 12:32:10 -0400] [35690] [INFO] Listening at: http://127.0.0.1:8000 (35690)
[2020-03-26 12:32:10 -0400] [35690] [INFO] Using worker: sync
[2020-03-26 12:32:10 -0400] [35693] [INFO] Booting worker with pid: 35693
[2020-03-26 12:32:10 -0400] [35695] [INFO] Booting worker with pid: 35695
[2020-03-26 12:32:10 -0400] [35696] [INFO] Booting worker with pid: 35696
[2020-03-26 12:32:10 -0400] [35697] [INFO] Booting worker with pid: 35697
[2020-03-26 12:32:10 -0400] [35690] [DEBUG] 4 workers
[2020-03-26 11:32:14 -0500] [35693] [DEBUG] OPTIONS /apidb/imagelist
[2020-03-26 11:32:14 -0500] [35693] [DEBUG] GET /apidb/imagelist
[2020-03-26 11:32:16 -0500] [35696] [DEBUG] OPTIONS /pics/california.PNG
[2020-03-26 11:32:16 -0500] [35693] [DEBUG] GET /pics/california.PNG
[2020-03-26 11:32:18 -0500] [35693] [DEBUG] GET /pics/california.PNG
[2020-03-26 11:32:20 -0500] [35693] [DEBUG] OPTIONS /pics/renewnables_1.JPG
[2020-03-26 11:32:20 -0500] [35693] [DEBUG] GET /pics/renewnables_1.JPG
[2020-03-26 11:32:21 -0500] [35695] [DEBUG] OPTIONS /pics/cyberpunk.JPG
[2020-03-26 11:32:21 -0500] [35697] [DEBUG] GET /pics/cyberpunk.JPG
[2020-03-26 12:32:26 -0400] [35690] [INFO] Handling signal: int
[2020-03-26 11:32:26 -0500] [35695] [INFO] Worker exiting (pid: 35695)
[2020-03-26 11:32:26 -0500] [35693] [INFO] Worker exiting (pid: 35693)
[2020-03-26 11:32:26 -0500] [35696] [INFO] Worker exiting (pid: 35696)
[2020-03-26 11:32:26 -0500] [35697] [INFO] Worker exiting (pid: 35697)
[2020-03-26 12:32:26 -0400] [35690] [INFO] Shutting down: Master
[2020-03-26 13:17:59 -0400] [35972] [DEBUG] Current configuration:
  config: gunicorn.conf.py
  bind: ['127.0.0.1:8000']
  backlog: 1024
  workers: 4
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  chdir: /Users/connorpro/Code/website_python
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 501
  group: 20
  umask: 0
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /Users/connorpro/Code/website_python/logs/gunicorn-access.log
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /Users/connorpro/Code/website_python/logs/gunicorn-error.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  syslog_addr: unix:///var/run/syslog
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  statsd_prefix: 
  proc_name: None
  default_proc_name: website_py2.wsgi
  django_settings: None
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x105a8b8c8>
  on_reload: <function OnReload.on_reload at 0x105a8ba60>
  when_ready: <function WhenReady.when_ready at 0x105a8bbf8>
  pre_fork: <function Prefork.pre_fork at 0x105a8bd90>
  post_fork: <function Postfork.post_fork at 0x105a8bf28>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x105a94158>
  worker_int: <function WorkerInt.worker_int at 0x105a942f0>
  worker_abort: <function WorkerAbort.worker_abort at 0x105a94488>
  pre_exec: <function PreExec.pre_exec at 0x105a94620>
  pre_request: <function PreRequest.pre_request at 0x105a947b8>
  post_request: <function PostRequest.post_request at 0x105a948c8>
  worker_exit: <function WorkerExit.worker_exit at 0x105a94a60>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x105a94bf8>
  on_exit: <function OnExit.on_exit at 0x105a94d90>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 3
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
[2020-03-26 13:17:59 -0400] [35972] [INFO] Starting gunicorn 19.6.0
[2020-03-26 13:17:59 -0400] [35972] [DEBUG] Arbiter booted
[2020-03-26 13:17:59 -0400] [35972] [INFO] Listening at: http://127.0.0.1:8000 (35972)
[2020-03-26 13:17:59 -0400] [35972] [INFO] Using worker: sync
[2020-03-26 13:17:59 -0400] [35975] [INFO] Booting worker with pid: 35975
[2020-03-26 13:17:59 -0400] [35976] [INFO] Booting worker with pid: 35976
[2020-03-26 13:17:59 -0400] [35977] [INFO] Booting worker with pid: 35977
[2020-03-26 13:17:59 -0400] [35978] [INFO] Booting worker with pid: 35978
[2020-03-26 13:17:59 -0400] [35972] [DEBUG] 4 workers
[2020-03-26 12:18:03 -0500] [35976] [DEBUG] GET /pics/renewnables_1.JPG
[2020-03-26 13:18:14 -0400] [35972] [INFO] Handling signal: int
[2020-03-26 12:18:14 -0500] [35976] [INFO] Worker exiting (pid: 35976)
[2020-03-26 12:18:14 -0500] [35977] [INFO] Worker exiting (pid: 35977)
[2020-03-26 12:18:14 -0500] [35978] [INFO] Worker exiting (pid: 35978)
[2020-03-26 12:18:14 -0500] [35975] [INFO] Worker exiting (pid: 35975)
[2020-03-26 13:18:14 -0400] [35972] [INFO] Shutting down: Master
[2020-03-26 13:18:26 -0400] [36003] [DEBUG] Current configuration:
  config: gunicorn.conf.py
  bind: ['127.0.0.1:8000']
  backlog: 1024
  workers: 4
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  chdir: /Users/connorpro/Code/website_python
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 501
  group: 20
  umask: 0
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /Users/connorpro/Code/website_python/logs/gunicorn-access.log
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /Users/connorpro/Code/website_python/logs/gunicorn-error.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  syslog_addr: unix:///var/run/syslog
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  statsd_prefix: 
  proc_name: None
  default_proc_name: website_py2.wsgi
  django_settings: None
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x102b1f8c8>
  on_reload: <function OnReload.on_reload at 0x102b1fa60>
  when_ready: <function WhenReady.when_ready at 0x102b1fbf8>
  pre_fork: <function Prefork.pre_fork at 0x102b1fd90>
  post_fork: <function Postfork.post_fork at 0x102b1ff28>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x102b28158>
  worker_int: <function WorkerInt.worker_int at 0x102b282f0>
  worker_abort: <function WorkerAbort.worker_abort at 0x102b28488>
  pre_exec: <function PreExec.pre_exec at 0x102b28620>
  pre_request: <function PreRequest.pre_request at 0x102b287b8>
  post_request: <function PostRequest.post_request at 0x102b288c8>
  worker_exit: <function WorkerExit.worker_exit at 0x102b28a60>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x102b28bf8>
  on_exit: <function OnExit.on_exit at 0x102b28d90>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 3
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
[2020-03-26 13:18:26 -0400] [36003] [INFO] Starting gunicorn 19.6.0
[2020-03-26 13:18:26 -0400] [36003] [DEBUG] Arbiter booted
[2020-03-26 13:18:26 -0400] [36003] [INFO] Listening at: http://127.0.0.1:8000 (36003)
[2020-03-26 13:18:26 -0400] [36003] [INFO] Using worker: sync
[2020-03-26 13:18:26 -0400] [36006] [INFO] Booting worker with pid: 36006
[2020-03-26 13:18:26 -0400] [36008] [INFO] Booting worker with pid: 36008
[2020-03-26 13:18:26 -0400] [36009] [INFO] Booting worker with pid: 36009
[2020-03-26 13:18:26 -0400] [36010] [INFO] Booting worker with pid: 36010
[2020-03-26 13:18:26 -0400] [36003] [DEBUG] 4 workers
[2020-03-26 12:18:34 -0500] [36009] [DEBUG] GET /pics/renewnables_2.JPG
[2020-03-26 13:18:47 -0400] [36003] [INFO] Handling signal: int
[2020-03-26 12:18:47 -0500] [36009] [INFO] Worker exiting (pid: 36009)
[2020-03-26 12:18:47 -0500] [36010] [INFO] Worker exiting (pid: 36010)
[2020-03-26 12:18:47 -0500] [36008] [INFO] Worker exiting (pid: 36008)
[2020-03-26 12:18:47 -0500] [36006] [INFO] Worker exiting (pid: 36006)
[2020-03-26 13:18:47 -0400] [36003] [INFO] Shutting down: Master
[2020-03-26 13:18:48 -0400] [36016] [DEBUG] Current configuration:
  config: gunicorn.conf.py
  bind: ['127.0.0.1:8000']
  backlog: 1024
  workers: 4
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  chdir: /Users/connorpro/Code/website_python
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 501
  group: 20
  umask: 0
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /Users/connorpro/Code/website_python/logs/gunicorn-access.log
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /Users/connorpro/Code/website_python/logs/gunicorn-error.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  syslog_addr: unix:///var/run/syslog
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  statsd_prefix: 
  proc_name: None
  default_proc_name: website_py2.wsgi
  django_settings: None
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x10c5c78c8>
  on_reload: <function OnReload.on_reload at 0x10c5c7a60>
  when_ready: <function WhenReady.when_ready at 0x10c5c7bf8>
  pre_fork: <function Prefork.pre_fork at 0x10c5c7d90>
  post_fork: <function Postfork.post_fork at 0x10c5c7f28>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x10c5d0158>
  worker_int: <function WorkerInt.worker_int at 0x10c5d02f0>
  worker_abort: <function WorkerAbort.worker_abort at 0x10c5d0488>
  pre_exec: <function PreExec.pre_exec at 0x10c5d0620>
  pre_request: <function PreRequest.pre_request at 0x10c5d07b8>
  post_request: <function PostRequest.post_request at 0x10c5d08c8>
  worker_exit: <function WorkerExit.worker_exit at 0x10c5d0a60>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x10c5d0bf8>
  on_exit: <function OnExit.on_exit at 0x10c5d0d90>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 3
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
[2020-03-26 13:18:48 -0400] [36016] [INFO] Starting gunicorn 19.6.0
[2020-03-26 13:18:48 -0400] [36016] [DEBUG] Arbiter booted
[2020-03-26 13:18:48 -0400] [36016] [INFO] Listening at: http://127.0.0.1:8000 (36016)
[2020-03-26 13:18:48 -0400] [36016] [INFO] Using worker: sync
[2020-03-26 13:18:48 -0400] [36019] [INFO] Booting worker with pid: 36019
[2020-03-26 13:18:48 -0400] [36020] [INFO] Booting worker with pid: 36020
[2020-03-26 13:18:48 -0400] [36022] [INFO] Booting worker with pid: 36022
[2020-03-26 13:18:48 -0400] [36023] [INFO] Booting worker with pid: 36023
[2020-03-26 13:18:48 -0400] [36016] [DEBUG] 4 workers
[2020-03-26 12:18:53 -0500] [36019] [DEBUG] OPTIONS /pics/officespace_1.JPG
[2020-03-26 12:18:53 -0500] [36023] [DEBUG] GET /pics/officespace_1.JPG
[2020-03-26 12:18:55 -0500] [36023] [DEBUG] OPTIONS /pics/spaceman.PNG
[2020-03-26 12:18:55 -0500] [36023] [DEBUG] GET /pics/spaceman.PNG
[2020-03-26 12:18:56 -0500] [36023] [DEBUG] GET /pics/spaceman.PNG
[2020-03-26 12:18:57 -0500] [36023] [DEBUG] GET /pics/spaceman.PNG
[2020-03-26 12:18:58 -0500] [36020] [DEBUG] OPTIONS /pics/surf.PNG
[2020-03-26 12:18:58 -0500] [36020] [DEBUG] GET /pics/surf.PNG
[2020-03-26 12:19:00 -0500] [36020] [DEBUG] GET /pics/surf.PNG
[2020-03-26 12:19:11 -0500] [36019] [DEBUG] OPTIONS /pics/charleston%20bridge.jpg
[2020-03-26 12:19:11 -0500] [36019] [DEBUG] GET /pics/charleston%20bridge.jpg
[2020-03-26 12:19:18 -0500] [36020] [DEBUG] GET /pics/spaceman.PNG
[2020-03-26 12:19:37 -0500] [36019] [DEBUG] GET /pics/officespace_1.JPG
[2020-03-26 12:20:04 -0500] [36023] [DEBUG] OPTIONS /pics/gradients_2.JPG
[2020-03-26 12:20:04 -0500] [36020] [DEBUG] GET /pics/gradients_2.JPG
[2020-03-26 12:22:45 -0500] [36022] [DEBUG] GET /pics/california.PNG
[2020-03-26 12:22:48 -0500] [36023] [DEBUG] GET /pics/house_2.JPG
[2020-03-26 12:22:49 -0500] [36022] [DEBUG] OPTIONS /pics/house_1.JPG
[2020-03-26 12:22:49 -0500] [36023] [DEBUG] GET /pics/house_1.JPG
[2020-03-26 12:22:50 -0500] [36019] [DEBUG] GET /pics/charleston%20bridge.jpg
[2020-03-26 12:22:53 -0500] [36019] [DEBUG] OPTIONS /pics/%3Cspan%20class=%22MuiButton-label%22%3Egradients_1.JPG%20%3C/span%3E%3Cspan%20class=%22MuiTouchRipple-root%22%3E%3Cspan%20class=%22MuiTouchRipple-ripple%20MuiTouchRipple-rippleVisible%22%20style=%22width:%20397.854px;%20height:%20397.854px;%20top:%20-165.927px;%20left:%20-31.9271px;%22%3E%3Cspan%20class=%22MuiTouchRipple-child%20MuiTouchRipple-childLeaving%22%3E%3C/span%3E%3C/span%3E%3C/span%3E
[2020-03-26 12:22:53 -0500] [36020] [DEBUG] GET /pics/%3Cspan%20class=%22MuiButton-label%22%3Egradients_1.JPG%20%3C/span%3E%3Cspan%20class=%22MuiTouchRipple-root%22%3E%3Cspan%20class=%22MuiTouchRipple-ripple%20MuiTouchRipple-rippleVisible%22%20style=%22width:%20397.854px;%20height:%20397.854px;%20top:%20-165.927px;%20left:%20-31.9271px;%22%3E%3Cspan%20class=%22MuiTouchRipple-child%20MuiTouchRipple-childLeaving%22%3E%3C/span%3E%3C/span%3E%3C/span%3E
[2020-03-26 12:22:54 -0500] [36023] [DEBUG] OPTIONS /pics/sanfran.JPG
[2020-03-26 12:22:54 -0500] [36020] [DEBUG] GET /pics/sanfran.JPG
[2020-03-26 12:24:15 -0500] [36020] [DEBUG] OPTIONS /pics/gradients_1.JPG
[2020-03-26 12:24:15 -0500] [36020] [DEBUG] GET /pics/gradients_1.JPG
[2020-03-26 13:25:18 -0400] [36016] [INFO] Handling signal: int
[2020-03-26 12:25:18 -0500] [36022] [INFO] Worker exiting (pid: 36022)
[2020-03-26 12:25:18 -0500] [36020] [INFO] Worker exiting (pid: 36020)
[2020-03-26 12:25:18 -0500] [36023] [INFO] Worker exiting (pid: 36023)
[2020-03-26 12:25:18 -0500] [36019] [INFO] Worker exiting (pid: 36019)
[2020-03-26 13:25:18 -0400] [36016] [INFO] Shutting down: Master
[2020-03-26 13:25:19 -0400] [36053] [INFO] Starting gunicorn 19.6.0
[2020-03-26 13:25:19 -0400] [36053] [INFO] Listening at: http://127.0.0.1:8000 (36053)
[2020-03-26 13:25:19 -0400] [36053] [INFO] Using worker: sync
[2020-03-26 13:25:19 -0400] [36056] [INFO] Booting worker with pid: 36056
[2020-03-26 13:25:19 -0400] [36058] [INFO] Booting worker with pid: 36058
[2020-03-26 13:25:19 -0400] [36059] [INFO] Booting worker with pid: 36059
[2020-03-26 13:25:19 -0400] [36060] [INFO] Booting worker with pid: 36060
[2020-03-26 13:52:34 -0400] [36053] [INFO] Handling signal: int
[2020-03-26 12:52:34 -0500] [36059] [INFO] Worker exiting (pid: 36059)
[2020-03-26 12:52:34 -0500] [36058] [INFO] Worker exiting (pid: 36058)
[2020-03-26 12:52:34 -0500] [36060] [INFO] Worker exiting (pid: 36060)
[2020-03-26 12:52:34 -0500] [36056] [INFO] Worker exiting (pid: 36056)
[2020-03-26 13:52:34 -0400] [36053] [INFO] Shutting down: Master
[2020-03-27 11:47:52 -0400] [37458] [DEBUG] Current configuration:
  config: gunicorn.conf.py
  bind: ['127.0.0.1:8000']
  backlog: 1024
  workers: 4
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  chdir: /Users/connorpro/Code/website_python
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 501
  group: 20
  umask: 0
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /Users/connorpro/Code/website_python/logs/gunicorn-access.log
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /Users/connorpro/Code/website_python/logs/gunicorn-error.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  syslog_addr: unix:///var/run/syslog
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  statsd_prefix: 
  proc_name: None
  default_proc_name: website_py2.wsgi
  django_settings: None
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x110e668c8>
  on_reload: <function OnReload.on_reload at 0x110e66a60>
  when_ready: <function WhenReady.when_ready at 0x110e66bf8>
  pre_fork: <function Prefork.pre_fork at 0x110e66d90>
  post_fork: <function Postfork.post_fork at 0x110e66f28>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x110e6f158>
  worker_int: <function WorkerInt.worker_int at 0x110e6f2f0>
  worker_abort: <function WorkerAbort.worker_abort at 0x110e6f488>
  pre_exec: <function PreExec.pre_exec at 0x110e6f620>
  pre_request: <function PreRequest.pre_request at 0x110e6f7b8>
  post_request: <function PostRequest.post_request at 0x110e6f8c8>
  worker_exit: <function WorkerExit.worker_exit at 0x110e6fa60>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x110e6fbf8>
  on_exit: <function OnExit.on_exit at 0x110e6fd90>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 3
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
[2020-03-27 11:47:52 -0400] [37458] [INFO] Starting gunicorn 19.6.0
[2020-03-27 11:47:52 -0400] [37458] [DEBUG] Arbiter booted
[2020-03-27 11:47:52 -0400] [37458] [INFO] Listening at: http://127.0.0.1:8000 (37458)
[2020-03-27 11:47:52 -0400] [37458] [INFO] Using worker: sync
[2020-03-27 11:47:52 -0400] [37461] [INFO] Booting worker with pid: 37461
[2020-03-27 11:47:52 -0400] [37463] [INFO] Booting worker with pid: 37463
[2020-03-27 11:47:52 -0400] [37464] [INFO] Booting worker with pid: 37464
[2020-03-27 11:47:53 -0400] [37465] [INFO] Booting worker with pid: 37465
[2020-03-27 11:47:53 -0400] [37458] [DEBUG] 4 workers
[2020-03-27 11:48:05 -0400] [37458] [INFO] Handling signal: int
[2020-03-27 10:48:05 -0500] [37464] [INFO] Worker exiting (pid: 37464)
[2020-03-27 10:48:05 -0500] [37463] [INFO] Worker exiting (pid: 37463)
[2020-03-27 10:48:05 -0500] [37465] [INFO] Worker exiting (pid: 37465)
[2020-03-27 10:48:05 -0500] [37461] [INFO] Worker exiting (pid: 37461)
[2020-03-27 11:48:06 -0400] [37458] [INFO] Shutting down: Master
[2020-04-04 17:08:04 -0400] [8850] [DEBUG] Current configuration:
  config: gunicorn.conf.py
  bind: ['127.0.0.1:8000']
  backlog: 1024
  workers: 4
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  chdir: /Users/connorpro/Code/website_python
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 501
  group: 20
  umask: 0
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /Users/connorpro/Code/website_python/logs/gunicorn-access.log
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /Users/connorpro/Code/website_python/logs/gunicorn-error.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  syslog_addr: unix:///var/run/syslog
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  statsd_prefix: 
  proc_name: None
  default_proc_name: website_py2.wsgi
  django_settings: None
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x107f048c8>
  on_reload: <function OnReload.on_reload at 0x107f04a60>
  when_ready: <function WhenReady.when_ready at 0x107f04bf8>
  pre_fork: <function Prefork.pre_fork at 0x107f04d90>
  post_fork: <function Postfork.post_fork at 0x107f04f28>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x107f0d158>
  worker_int: <function WorkerInt.worker_int at 0x107f0d2f0>
  worker_abort: <function WorkerAbort.worker_abort at 0x107f0d488>
  pre_exec: <function PreExec.pre_exec at 0x107f0d620>
  pre_request: <function PreRequest.pre_request at 0x107f0d7b8>
  post_request: <function PostRequest.post_request at 0x107f0d8c8>
  worker_exit: <function WorkerExit.worker_exit at 0x107f0da60>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x107f0dbf8>
  on_exit: <function OnExit.on_exit at 0x107f0dd90>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 3
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
[2020-04-04 17:08:04 -0400] [8850] [INFO] Starting gunicorn 19.6.0
[2020-04-04 17:08:04 -0400] [8850] [DEBUG] Arbiter booted
[2020-04-04 17:08:04 -0400] [8850] [INFO] Listening at: http://127.0.0.1:8000 (8850)
[2020-04-04 17:08:04 -0400] [8850] [INFO] Using worker: sync
[2020-04-04 17:08:04 -0400] [8853] [INFO] Booting worker with pid: 8853
[2020-04-04 17:08:04 -0400] [8854] [INFO] Booting worker with pid: 8854
[2020-04-04 17:08:04 -0400] [8856] [INFO] Booting worker with pid: 8856
[2020-04-04 17:08:04 -0400] [8857] [INFO] Booting worker with pid: 8857
[2020-04-04 17:08:04 -0400] [8850] [DEBUG] 4 workers
[2020-04-04 16:08:14 -0500] [8853] [DEBUG] GET /apidb/ws/chat
[2020-04-04 16:08:14 -0500] [8857] [DEBUG] GET /apidb/comments
[2020-04-04 16:12:40 -0500] [8857] [DEBUG] GET /apidb/ws/chat
[2020-04-04 16:12:40 -0500] [8856] [DEBUG] GET /apidb/comments
[2020-04-04 16:22:57 -0500] [8857] [DEBUG] GET /apidb/ws/chat
[2020-04-04 16:22:57 -0500] [8856] [DEBUG] GET /apidb/comments
[2020-04-04 16:27:00 -0500] [8857] [DEBUG] GET /apidb/ws/chat
[2020-04-04 16:27:00 -0500] [8856] [DEBUG] GET /apidb/comments
[2020-04-04 16:27:05 -0500] [8854] [DEBUG] GET /apidb/ws/chat
[2020-04-04 16:27:05 -0500] [8857] [DEBUG] GET /apidb/comments
[2020-04-04 17:27:11 -0400] [8850] [INFO] Handling signal: int
[2020-04-04 16:27:11 -0500] [8857] [INFO] Worker exiting (pid: 8857)
[2020-04-04 16:27:11 -0500] [8853] [INFO] Worker exiting (pid: 8853)
[2020-04-04 16:27:11 -0500] [8856] [INFO] Worker exiting (pid: 8856)
[2020-04-04 16:27:11 -0500] [8854] [INFO] Worker exiting (pid: 8854)
[2020-04-04 17:27:11 -0400] [8850] [INFO] Shutting down: Master
[2020-04-04 17:27:31 -0400] [8982] [DEBUG] Current configuration:
  config: gunicorn.conf.py
  bind: ['127.0.0.1:8000']
  backlog: 1024
  workers: 4
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  chdir: /Users/connorpro/Code/website_python
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 501
  group: 20
  umask: 0
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /Users/connorpro/Code/website_python/logs/gunicorn-access.log
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /Users/connorpro/Code/website_python/logs/gunicorn-error.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  syslog_addr: unix:///var/run/syslog
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  statsd_prefix: 
  proc_name: None
  default_proc_name: website_py2.wsgi
  django_settings: None
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x106ac68c8>
  on_reload: <function OnReload.on_reload at 0x106ac6a60>
  when_ready: <function WhenReady.when_ready at 0x106ac6bf8>
  pre_fork: <function Prefork.pre_fork at 0x106ac6d90>
  post_fork: <function Postfork.post_fork at 0x106ac6f28>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x106acf158>
  worker_int: <function WorkerInt.worker_int at 0x106acf2f0>
  worker_abort: <function WorkerAbort.worker_abort at 0x106acf488>
  pre_exec: <function PreExec.pre_exec at 0x106acf620>
  pre_request: <function PreRequest.pre_request at 0x106acf7b8>
  post_request: <function PostRequest.post_request at 0x106acf8c8>
  worker_exit: <function WorkerExit.worker_exit at 0x106acfa60>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x106acfbf8>
  on_exit: <function OnExit.on_exit at 0x106acfd90>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 3
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
[2020-04-04 17:27:31 -0400] [8982] [INFO] Starting gunicorn 19.6.0
[2020-04-04 17:27:31 -0400] [8982] [DEBUG] Arbiter booted
[2020-04-04 17:27:31 -0400] [8982] [INFO] Listening at: http://127.0.0.1:8000 (8982)
[2020-04-04 17:27:31 -0400] [8982] [INFO] Using worker: sync
[2020-04-04 17:27:31 -0400] [8985] [INFO] Booting worker with pid: 8985
[2020-04-04 17:27:31 -0400] [8987] [INFO] Booting worker with pid: 8987
[2020-04-04 17:27:31 -0400] [8988] [INFO] Booting worker with pid: 8988
[2020-04-04 17:27:31 -0400] [8989] [INFO] Booting worker with pid: 8989
[2020-04-04 17:27:31 -0400] [8982] [DEBUG] 4 workers
[2020-04-04 17:43:19 -0400] [8982] [INFO] Handling signal: int
[2020-04-04 16:43:19 -0500] [8989] [INFO] Worker exiting (pid: 8989)
[2020-04-04 16:43:19 -0500] [8988] [INFO] Worker exiting (pid: 8988)
[2020-04-04 16:43:19 -0500] [8987] [INFO] Worker exiting (pid: 8987)
[2020-04-04 16:43:19 -0500] [8985] [INFO] Worker exiting (pid: 8985)
[2020-04-04 17:43:19 -0400] [8982] [INFO] Shutting down: Master
[2020-04-04 17:43:20 -0400] [9069] [DEBUG] Current configuration:
  config: gunicorn.conf.py
  bind: ['127.0.0.1:8000']
  backlog: 1024
  workers: 1
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  chdir: /Users/connorpro/Code/website_python
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 501
  group: 20
  umask: 0
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /Users/connorpro/Code/website_python/logs/gunicorn-access.log
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /Users/connorpro/Code/website_python/logs/gunicorn-error.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  syslog_addr: unix:///var/run/syslog
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  statsd_prefix: 
  proc_name: None
  default_proc_name: website_py2.wsgi
  django_settings: None
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x1035648c8>
  on_reload: <function OnReload.on_reload at 0x103564a60>
  when_ready: <function WhenReady.when_ready at 0x103564bf8>
  pre_fork: <function Prefork.pre_fork at 0x103564d90>
  post_fork: <function Postfork.post_fork at 0x103564f28>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x10356d158>
  worker_int: <function WorkerInt.worker_int at 0x10356d2f0>
  worker_abort: <function WorkerAbort.worker_abort at 0x10356d488>
  pre_exec: <function PreExec.pre_exec at 0x10356d620>
  pre_request: <function PreRequest.pre_request at 0x10356d7b8>
  post_request: <function PostRequest.post_request at 0x10356d8c8>
  worker_exit: <function WorkerExit.worker_exit at 0x10356da60>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x10356dbf8>
  on_exit: <function OnExit.on_exit at 0x10356dd90>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 3
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
[2020-04-04 17:43:20 -0400] [9069] [INFO] Starting gunicorn 19.6.0
[2020-04-04 17:43:20 -0400] [9069] [DEBUG] Arbiter booted
[2020-04-04 17:43:20 -0400] [9069] [INFO] Listening at: http://127.0.0.1:8000 (9069)
[2020-04-04 17:43:20 -0400] [9069] [INFO] Using worker: sync
[2020-04-04 17:43:20 -0400] [9072] [INFO] Booting worker with pid: 9072
[2020-04-04 17:43:20 -0400] [9069] [DEBUG] 1 workers
[2020-04-04 16:43:25 -0500] [9072] [DEBUG] GET /apidb/ws/chat
[2020-04-04 16:43:25 -0500] [9072] [DEBUG] GET /apidb/comments
[2020-04-04 17:43:43 -0400] [9069] [INFO] Handling signal: int
[2020-04-04 16:43:43 -0500] [9072] [INFO] Worker exiting (pid: 9072)
[2020-04-04 17:43:43 -0400] [9069] [INFO] Shutting down: Master
[2020-04-04 17:43:45 -0400] [9075] [DEBUG] Current configuration:
  config: gunicorn.conf.py
  bind: ['127.0.0.1:8000']
  backlog: 1024
  workers: 1
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  chdir: /Users/connorpro/Code/website_python
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 501
  group: 20
  umask: 0
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /Users/connorpro/Code/website_python/logs/gunicorn-access.log
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /Users/connorpro/Code/website_python/logs/gunicorn-error.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  syslog_addr: unix:///var/run/syslog
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  statsd_prefix: 
  proc_name: None
  default_proc_name: website_py2.wsgi
  django_settings: None
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x10bb398c8>
  on_reload: <function OnReload.on_reload at 0x10bb39a60>
  when_ready: <function WhenReady.when_ready at 0x10bb39bf8>
  pre_fork: <function Prefork.pre_fork at 0x10bb39d90>
  post_fork: <function Postfork.post_fork at 0x10bb39f28>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x10bb42158>
  worker_int: <function WorkerInt.worker_int at 0x10bb422f0>
  worker_abort: <function WorkerAbort.worker_abort at 0x10bb42488>
  pre_exec: <function PreExec.pre_exec at 0x10bb42620>
  pre_request: <function PreRequest.pre_request at 0x10bb427b8>
  post_request: <function PostRequest.post_request at 0x10bb428c8>
  worker_exit: <function WorkerExit.worker_exit at 0x10bb42a60>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x10bb42bf8>
  on_exit: <function OnExit.on_exit at 0x10bb42d90>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 3
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: TLSv1
[2020-04-04 17:43:45 -0400] [9075] [INFO] Starting gunicorn 19.6.0
[2020-04-04 17:43:45 -0400] [9075] [DEBUG] Arbiter booted
[2020-04-04 17:43:45 -0400] [9075] [INFO] Listening at: http://127.0.0.1:8000 (9075)
[2020-04-04 17:43:45 -0400] [9075] [INFO] Using worker: sync
[2020-04-04 17:43:45 -0400] [9078] [INFO] Booting worker with pid: 9078
[2020-04-04 17:43:45 -0400] [9075] [DEBUG] 1 workers
[2020-04-04 16:43:54 -0500] [9078] [DEBUG] GET /apidb/ws/chat
[2020-04-04 16:43:54 -0500] [9078] [DEBUG] GET /apidb/comments
[2020-04-04 18:02:49 -0400] [9075] [INFO] Handling signal: int
[2020-04-04 17:02:49 -0500] [9078] [INFO] Worker exiting (pid: 9078)
[2020-04-04 18:02:49 -0400] [9075] [INFO] Shutting down: Master
